declare -x AZ_BATCH_ACCOUNT_NAME="exp1hildebatch"
declare -x AZ_BATCH_ACCOUNT_URL="https://exp1hildebatch.southeastasia.batch.azure.com/"
declare -x AZ_BATCH_CERTIFICATES_DIR="/mnt/batch/tasks/workitems/svm/job-1/task-00020/certs"
declare -x AZ_BATCH_JOB_ID="svm"
declare -x AZ_BATCH_JOB_PREP_DIR="/mnt/batch/tasks/workitems/svm/job-1/jobpreparation"
declare -x AZ_BATCH_JOB_PREP_WORKING_DIR="/mnt/batch/tasks/workitems/svm/job-1/jobpreparation/wd"
declare -x AZ_BATCH_NODE_ID="tvm-2112069986_7-20180411t200903z"
declare -x AZ_BATCH_NODE_IS_DEDICATED="true"
declare -x AZ_BATCH_NODE_ROOT_DIR="/mnt/batch/tasks"
declare -x AZ_BATCH_NODE_SHARED_DIR="/mnt/batch/tasks/shared"
declare -x AZ_BATCH_NODE_STARTUP_DIR="/mnt/batch/tasks/startup"
declare -x AZ_BATCH_POOL_ID="experiment"
declare -x AZ_BATCH_TASK_DIR="/mnt/batch/tasks/workitems/svm/job-1/task-00020"
declare -x AZ_BATCH_TASK_ID="task-00020"
declare -x AZ_BATCH_TASK_USER="_azbatch"
declare -x AZ_BATCH_TASK_USER_IDENTITY="PoolAdmin"
declare -x AZ_BATCH_TASK_WORKING_DIR="/mnt/batch/tasks/workitems/svm/job-1/task-00020/wd"
declare -x HOME="/mnt/batch/tasks/workitems/svm/job-1/task-00020/wd"
declare -x OLDPWD
declare -x PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/mnt/batch/tasks/shared:/mnt/batch/tasks/workitems/svm/job-1/task-00020/wd"
declare -x PWD="/mnt/batch/tasks/workitems/svm/job-1/task-00020/wd"
declare -x SHLVL="1"
declare -x USER="_azbatch"
[2018-04-11 21:21:34] Results will not be logged locally.
[2018-04-11 21:21:34] Retrieving tasks...
[2018-04-11 21:21:34] Tasks will be retrieved based on provided task ids.
[2018-04-11 21:21:34] The following tasks were retrieved: [9952].
[2018-04-11 21:21:41] Downloaded task 9952.
[2018-04-11 21:21:41] Prepared pipelines.
[2018-04-11 21:21:41] Started run 1 on task 9952, dataset 'phoneme'.
[2018-04-11 21:21:41] Started run 8 on task 9952, dataset 'phoneme'.
[2018-04-11 21:21:41] Started run 6 on task 9952, dataset 'phoneme'.
[2018-04-11 21:21:41] Started run 2 on task 9952, dataset 'phoneme'.
[2018-04-11 21:21:41] Started run 3 on task 9952, dataset 'phoneme'.
[2018-04-11 21:21:41] Started run 4 on task 9952, dataset 'phoneme'.
[2018-04-11 21:21:41] Started run 5 on task 9952, dataset 'phoneme'.
[2018-04-11 21:21:41] Started run 7 on task 9952, dataset 'phoneme'.
[2018-04-11 21:21:46] Error in run 8: Run already exists in server. Run id(s): {8987783}
[2018-04-11 21:21:46] Started run 9 on task 9952, dataset 'phoneme'.
[2018-04-11 21:21:46] Error in run 6: Run already exists in server. Run id(s): {8987772}
[2018-04-11 21:21:46] Error in run 1: Run already exists in server. Run id(s): {8941411}
[2018-04-11 21:21:46] Started run 10 on task 9952, dataset 'phoneme'.
[2018-04-11 21:21:46] Error in run 2: Run already exists in server. Run id(s): {8987781}
[2018-04-11 21:21:46] Started run 11 on task 9952, dataset 'phoneme'.
[2018-04-11 21:21:46] Error in run 3: Run already exists in server. Run id(s): {8987760}
[2018-04-11 21:21:46] Started run 12 on task 9952, dataset 'phoneme'.
[2018-04-11 21:21:46] Started run 13 on task 9952, dataset 'phoneme'.
[2018-04-11 21:21:46] Error in run 4: Run already exists in server. Run id(s): {8987768}
[2018-04-11 21:21:46] Started run 14 on task 9952, dataset 'phoneme'.
[2018-04-11 21:21:46] Error in run 5: Run already exists in server. Run id(s): {8987759}
[2018-04-11 21:21:46] Started run 15 on task 9952, dataset 'phoneme'.
[2018-04-11 21:21:46] Error in run 7: Run already exists in server. Run id(s): {8987782}
[2018-04-11 21:21:46] Started run 16 on task 9952, dataset 'phoneme'.
[2018-04-11 21:21:50] Error in run 11: Run already exists in server. Run id(s): {8987795}
[2018-04-11 21:21:50] Error in run 12: Run already exists in server. Run id(s): {8987794}
[2018-04-11 21:21:50] Error in run 9: Run already exists in server. Run id(s): {8987788}
[2018-04-11 21:21:50] Error in run 13: Run already exists in server. Run id(s): {8987802}
[2018-04-11 21:21:50] Error in run 10: Run already exists in server. Run id(s): {8987813}
[2018-04-11 21:21:50] Started run 17 on task 9952, dataset 'phoneme'.
[2018-04-11 21:21:50] Error in run 15: Run already exists in server. Run id(s): {8987824}
[2018-04-11 21:21:50] Error in run 14: Run already exists in server. Run id(s): {8987806}
[2018-04-11 21:21:50] Started run 18 on task 9952, dataset 'phoneme'.
[2018-04-11 21:21:50] Started run 22 on task 9952, dataset 'phoneme'.
[2018-04-11 21:21:50] Started run 19 on task 9952, dataset 'phoneme'.
[2018-04-11 21:21:50] Started run 23 on task 9952, dataset 'phoneme'.
[2018-04-11 21:21:50] Started run 20 on task 9952, dataset 'phoneme'.
[2018-04-11 21:21:50] Error in run 16: Run already exists in server. Run id(s): {8987838}
[2018-04-11 21:21:50] Started run 24 on task 9952, dataset 'phoneme'.
[2018-04-11 21:21:50] Started run 21 on task 9952, dataset 'phoneme'.
[2018-04-11 21:21:55] Error in run 17: Run already exists in server. Run id(s): {8987829}
[2018-04-11 21:21:55] Error in run 22: Run already exists in server. Run id(s): {8987857}
[2018-04-11 21:21:55] Started run 25 on task 9952, dataset 'phoneme'.
[2018-04-11 21:21:55] Error in run 18: Run already exists in server. Run id(s): {8987823}
[2018-04-11 21:21:55] Started run 26 on task 9952, dataset 'phoneme'.
[2018-04-11 21:21:55] Started run 27 on task 9952, dataset 'phoneme'.
[2018-04-11 21:21:55] Error in run 19: Run already exists in server. Run id(s): {8987836}
[2018-04-11 21:21:55] Error in run 24: Run already exists in server. Run id(s): {8987853}
[2018-04-11 21:21:55] Started run 28 on task 9952, dataset 'phoneme'.
[2018-04-11 21:21:55] Error in run 20: Run already exists in server. Run id(s): {8987837}
[2018-04-11 21:21:55] Started run 29 on task 9952, dataset 'phoneme'.
[2018-04-11 21:21:55] Started run 30 on task 9952, dataset 'phoneme'.
[2018-04-11 21:21:55] Error in run 21: Run already exists in server. Run id(s): {8987846}
[2018-04-11 21:21:55] Started run 31 on task 9952, dataset 'phoneme'.
[2018-04-11 21:21:55] Error in run 23: Run already exists in server. Run id(s): {8987851}
[2018-04-11 21:21:55] Started run 32 on task 9952, dataset 'phoneme'.
[2018-04-11 21:21:59] Error in run 26: Run already exists in server. Run id(s): {8987864}
[2018-04-11 21:21:59] Started run 33 on task 9952, dataset 'phoneme'.
[2018-04-11 21:21:59] Error in run 25: Run already exists in server. Run id(s): {8987868}
[2018-04-11 21:21:59] Error in run 27: Run already exists in server. Run id(s): {8987863}
[2018-04-11 21:21:59] Started run 34 on task 9952, dataset 'phoneme'.
[2018-04-11 21:21:59] Started run 35 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:00] Error in run 28: Run already exists in server. Run id(s): {8987869}
[2018-04-11 21:22:00] Started run 36 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:00] Error in run 29: Run already exists in server. Run id(s): {8987888}
[2018-04-11 21:22:00] Started run 37 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:00] Error in run 31: Run already exists in server. Run id(s): {8987879}
[2018-04-11 21:22:00] Error in run 30: Run already exists in server. Run id(s): {8987877}
[2018-04-11 21:22:00] Started run 38 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:00] Error in run 32: Run already exists in server. Run id(s): {8987884}
[2018-04-11 21:22:00] Started run 39 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:00] Started run 40 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:04] Error in run 34: Run already exists in server. Run id(s): {8987892}
[2018-04-11 21:22:04] Started run 41 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:04] Error in run 35: Run already exists in server. Run id(s): {8987893}
[2018-04-11 21:22:04] Error in run 33: Run already exists in server. Run id(s): {8987887}
[2018-04-11 21:22:04] Started run 42 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:04] Started run 43 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:04] Error in run 36: Run already exists in server. Run id(s): {8987908}
[2018-04-11 21:22:04] Started run 44 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:04] Error in run 37: Run already exists in server. Run id(s): {8987899}
[2018-04-11 21:22:04] Started run 45 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:04] Error in run 38: Run already exists in server. Run id(s): {8987900}
[2018-04-11 21:22:04] Started run 46 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:04] Error in run 39: Run already exists in server. Run id(s): {8987906}
[2018-04-11 21:22:04] Started run 47 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:05] Error in run 40: Run already exists in server. Run id(s): {8987913}
[2018-04-11 21:22:05] Started run 48 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:08] Error in run 43: Run already exists in server. Run id(s): {8987921}
[2018-04-11 21:22:08] Error in run 41: Run already exists in server. Run id(s): {8987915}
[2018-04-11 21:22:08] Started run 49 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:08] Started run 50 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:09] Error in run 42: Run already exists in server. Run id(s): {8987918}
[2018-04-11 21:22:09] Error in run 44: Run already exists in server. Run id(s): {8987943}
[2018-04-11 21:22:09] Started run 51 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:09] Started run 52 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:09] Error in run 45: Run already exists in server. Run id(s): {8987927}
[2018-04-11 21:22:09] Started run 53 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:09] Error in run 46: Run already exists in server. Run id(s): {8987928}
[2018-04-11 21:22:09] Started run 54 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:09] Error in run 47: Run already exists in server. Run id(s): {8987936}
[2018-04-11 21:22:09] Started run 55 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:09] Error in run 48: Run already exists in server. Run id(s): {8987937}
[2018-04-11 21:22:09] Started run 56 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:13] Error in run 49: Run already exists in server. Run id(s): {8987950}
[2018-04-11 21:22:13] Started run 57 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:13] Error in run 50: Run already exists in server. Run id(s): {8987952}
[2018-04-11 21:22:13] Started run 58 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:13] Error in run 51: Run already exists in server. Run id(s): {8987948}
[2018-04-11 21:22:13] Error in run 53: Run already exists in server. Run id(s): {8987951}
[2018-04-11 21:22:13] Error in run 52: Run already exists in server. Run id(s): {8987953}
[2018-04-11 21:22:13] Started run 59 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:13] Error in run 54: Run already exists in server. Run id(s): {8987956}
[2018-04-11 21:22:13] Started run 60 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:13] Started run 62 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:13] Started run 61 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:14] Error in run 55: Run already exists in server. Run id(s): {8987955}
[2018-04-11 21:22:14] Started run 63 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:14] Error in run 56: Run already exists in server. Run id(s): {8987957}
[2018-04-11 21:22:14] Started run 64 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:18] Error in run 57: Run already exists in server. Run id(s): {8987958}
[2018-04-11 21:22:18] Started run 65 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:18] Error in run 58: Run already exists in server. Run id(s): {8987959}
[2018-04-11 21:22:18] Started run 66 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:18] Error in run 61: Run already exists in server. Run id(s): {8987970}
[2018-04-11 21:22:18] Error in run 60: Run already exists in server. Run id(s): {8987962}
[2018-04-11 21:22:18] Started run 67 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:18] Started run 68 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:18] Error in run 59: Run already exists in server. Run id(s): {8987960}
[2018-04-11 21:22:18] Error in run 62: Run already exists in server. Run id(s): {8987989}
[2018-04-11 21:22:18] Started run 69 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:18] Started run 70 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:18] Error in run 63: Run already exists in server. Run id(s): {8987965}
[2018-04-11 21:22:18] Error in run 64: Run already exists in server. Run id(s): {8987966}
[2018-04-11 21:22:18] Started run 71 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:18] Started run 72 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:22] Error in run 65: Run already exists in server. Run id(s): {8987972}
[2018-04-11 21:22:22] Started run 73 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:22] Error in run 66: Run already exists in server. Run id(s): {8987974}
[2018-04-11 21:22:22] Started run 74 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:22] Error in run 67: Run already exists in server. Run id(s): {8987975}
[2018-04-11 21:22:22] Error in run 68: Run already exists in server. Run id(s): {8987982}
[2018-04-11 21:22:22] Started run 75 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:22] Started run 76 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:23] Error in run 69: Run already exists in server. Run id(s): {8987983}
[2018-04-11 21:22:23] Started run 77 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:23] Error in run 70: Run already exists in server. Run id(s): {8987986}
[2018-04-11 21:22:23] Started run 78 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:23] Error in run 71: Run already exists in server. Run id(s): {8987993}
[2018-04-11 21:22:23] Started run 79 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:23] Error in run 72: Run already exists in server. Run id(s): {8987994}
[2018-04-11 21:22:23] Started run 80 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:27] Error in run 73: Run already exists in server. Run id(s): {8987995}
[2018-04-11 21:22:27] Error in run 74: Run already exists in server. Run id(s): {8987998}
[2018-04-11 21:22:27] Started run 81 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:27] Started run 82 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:27] Error in run 76: Run already exists in server. Run id(s): {8988002}
[2018-04-11 21:22:27] Started run 83 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:27] Error in run 75: Run already exists in server. Run id(s): {8987999}
[2018-04-11 21:22:27] Started run 84 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:27] Error in run 77: Run already exists in server. Run id(s): {8988001}
[2018-04-11 21:22:27] Started run 85 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:28] Error in run 79: Run already exists in server. Run id(s): {8988005}
[2018-04-11 21:22:28] Error in run 80: Run already exists in server. Run id(s): {8988007}
[2018-04-11 21:22:28] Error in run 78: Run already exists in server. Run id(s): {8988003}
[2018-04-11 21:22:28] Started run 86 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:28] Started run 87 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:28] Started run 88 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:31] Error in run 81: Run already exists in server. Run id(s): {8988006}
[2018-04-11 21:22:31] Error in run 82: Run already exists in server. Run id(s): {8988008}
[2018-04-11 21:22:31] Started run 89 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:31] Started run 90 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:31] Error in run 84: Run already exists in server. Run id(s): {8988013}
[2018-04-11 21:22:31] Error in run 83: Run already exists in server. Run id(s): {8988009}
[2018-04-11 21:22:31] Started run 91 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:31] Started run 92 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:32] Error in run 85: Run already exists in server. Run id(s): {8988015}
[2018-04-11 21:22:32] Started run 93 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:32] Error in run 86: Run already exists in server. Run id(s): {8988012}
[2018-04-11 21:22:32] Started run 94 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:32] Error in run 87: Run already exists in server. Run id(s): {8988030}
[2018-04-11 21:22:32] Started run 95 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:32] Error in run 88: Run already exists in server. Run id(s): {8988019}
[2018-04-11 21:22:32] Started run 96 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:36] Error in run 89: Run already exists in server. Run id(s): {8988020}
[2018-04-11 21:22:36] Started run 97 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:36] Error in run 90: Run already exists in server. Run id(s): {8988021}
[2018-04-11 21:22:36] Started run 98 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:36] Error in run 91: Run already exists in server. Run id(s): {8988022}
[2018-04-11 21:22:36] Started run 99 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:36] Error in run 92: Run already exists in server. Run id(s): {8988043}
[2018-04-11 21:22:36] Started run 100 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:36] Error in run 93: Run already exists in server. Run id(s): {8988025}
[2018-04-11 21:22:36] Started run 101 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:36] Error in run 94: Run already exists in server. Run id(s): {8988027}
[2018-04-11 21:22:36] Started run 102 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:37] Error in run 95: Run already exists in server. Run id(s): {8988046}
[2018-04-11 21:22:37] Error in run 96: Run already exists in server. Run id(s): {8988032}
[2018-04-11 21:22:37] Started run 103 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:37] Started run 104 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:40] Error in run 97: Run already exists in server. Run id(s): {8988036}
[2018-04-11 21:22:40] Started run 105 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:40] Error in run 98: Run already exists in server. Run id(s): {8988041}
[2018-04-11 21:22:40] Started run 106 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:40] Error in run 99: Run already exists in server. Run id(s): {8988045}
[2018-04-11 21:22:40] Started run 107 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:40] Error in run 100: Run already exists in server. Run id(s): {8988048}
[2018-04-11 21:22:40] Started run 108 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:41] Error in run 101: Run already exists in server. Run id(s): {8988047}
[2018-04-11 21:22:41] Error in run 102: Run already exists in server. Run id(s): {8988049}
[2018-04-11 21:22:41] Started run 109 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:41] Started run 110 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:41] Error in run 103: Run already exists in server. Run id(s): {8988055}
[2018-04-11 21:22:41] Started run 111 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:41] Error in run 104: Run already exists in server. Run id(s): {8988052}
[2018-04-11 21:22:41] Started run 112 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:45] Error in run 105: Run already exists in server. Run id(s): {8988053}
[2018-04-11 21:22:45] Error in run 106: Run already exists in server. Run id(s): {8988056}
[2018-04-11 21:22:45] Error in run 107: Run already exists in server. Run id(s): {8988059}
[2018-04-11 21:22:45] Started run 113 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:45] Started run 114 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:45] Error in run 108: Run already exists in server. Run id(s): {8988063}
[2018-04-11 21:22:45] Started run 116 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:45] Started run 115 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:45] Error in run 109: Run already exists in server. Run id(s): {8988071}
[2018-04-11 21:22:45] Started run 117 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:46] Error in run 110: Run already exists in server. Run id(s): {8988061}
[2018-04-11 21:22:46] Started run 118 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:46] Error in run 111: Run already exists in server. Run id(s): {8988064}
[2018-04-11 21:22:46] Started run 119 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:46] Error in run 112: Run already exists in server. Run id(s): {8988070}
[2018-04-11 21:22:46] Started run 120 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:49] Error in run 113: Run already exists in server. Run id(s): {8988068}
[2018-04-11 21:22:49] Started run 121 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:50] Error in run 116: Run already exists in server. Run id(s): {8988074}
[2018-04-11 21:22:50] Error in run 114: Run already exists in server. Run id(s): {8988067}
[2018-04-11 21:22:50] Started run 122 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:50] Started run 123 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:50] Error in run 115: Run already exists in server. Run id(s): {8988073}
[2018-04-11 21:22:50] Started run 124 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:50] Error in run 117: Run already exists in server. Run id(s): {8988075}
[2018-04-11 21:22:50] Started run 125 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:50] Error in run 118: Run already exists in server. Run id(s): {8988076}
[2018-04-11 21:22:50] Started run 126 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:50] Error in run 120: Run already exists in server. Run id(s): {8988077}
[2018-04-11 21:22:50] Started run 127 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:50] Error in run 119: Run already exists in server. Run id(s): {8988080}
[2018-04-11 21:22:50] Started run 128 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:54] Error in run 121: Run already exists in server. Run id(s): {8988078}
[2018-04-11 21:22:54] Started run 129 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:54] Error in run 122: Run already exists in server. Run id(s): {8988079}
[2018-04-11 21:22:54] Started run 130 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:54] Error in run 123: Run already exists in server. Run id(s): {8988088}
[2018-04-11 21:22:54] Started run 131 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:54] Error in run 124: Run already exists in server. Run id(s): {8988097}
[2018-04-11 21:22:54] Started run 132 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:54] Error in run 125: Run already exists in server. Run id(s): {8988084}
[2018-04-11 21:22:54] Started run 133 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:55] Error in run 126: Run already exists in server. Run id(s): {8988083}
[2018-04-11 21:22:55] Started run 134 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:55] Error in run 128: Run already exists in server. Run id(s): {8988087}
[2018-04-11 21:22:55] Started run 135 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:55] Error in run 127: Run already exists in server. Run id(s): {8988085}
[2018-04-11 21:22:55] Started run 136 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:58] Error in run 129: Run already exists in server. Run id(s): {8988091}
[2018-04-11 21:22:58] Started run 137 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:58] Error in run 130: Run already exists in server. Run id(s): {8988101}
[2018-04-11 21:22:58] Started run 138 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:58] Error in run 131: Run already exists in server. Run id(s): {8988102}
[2018-04-11 21:22:58] Started run 139 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:59] Error in run 133: Run already exists in server. Run id(s): {8988108}
[2018-04-11 21:22:59] Error in run 132: Run already exists in server. Run id(s): {8988104}
[2018-04-11 21:22:59] Started run 140 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:59] Started run 141 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:59] Error in run 134: Run already exists in server. Run id(s): {8988105}
[2018-04-11 21:22:59] Started run 142 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:59] Error in run 135: Run already exists in server. Run id(s): {8988109}
[2018-04-11 21:22:59] Error in run 136: Run already exists in server. Run id(s): {8988113}
[2018-04-11 21:22:59] Started run 143 on task 9952, dataset 'phoneme'.
[2018-04-11 21:22:59] Started run 144 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:03] Error in run 137: Run already exists in server. Run id(s): {8988116}
[2018-04-11 21:23:03] Started run 145 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:03] Error in run 138: Run already exists in server. Run id(s): {8988128}
[2018-04-11 21:23:03] Started run 146 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:03] Error in run 139: Run already exists in server. Run id(s): {8988118}
[2018-04-11 21:23:03] Started run 147 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:04] Error in run 140: Run already exists in server. Run id(s): {8988121}
[2018-04-11 21:23:04] Started run 148 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:04] Error in run 141: Run already exists in server. Run id(s): {8988122}
[2018-04-11 21:23:04] Started run 149 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:04] Error in run 142: Run already exists in server. Run id(s): {8988123}
[2018-04-11 21:23:04] Started run 150 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:04] Error in run 143: Run already exists in server. Run id(s): {8988126}
[2018-04-11 21:23:04] Started run 151 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:04] Error in run 144: Run already exists in server. Run id(s): {8988129}
[2018-04-11 21:23:04] Started run 152 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:08] Error in run 145: Run already exists in server. Run id(s): {8988127}
[2018-04-11 21:23:08] Started run 153 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:08] Error in run 146: Run already exists in server. Run id(s): {8988133}
[2018-04-11 21:23:08] Started run 154 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:08] Error in run 147: Run already exists in server. Run id(s): {8988132}
[2018-04-11 21:23:08] Started run 155 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:08] Error in run 148: Run already exists in server. Run id(s): {8988135}
[2018-04-11 21:23:08] Started run 156 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:09] Error in run 150: Run already exists in server. Run id(s): {8988138}
[2018-04-11 21:23:09] Error in run 149: Run already exists in server. Run id(s): {8988146}
[2018-04-11 21:23:09] Error in run 152: Run already exists in server. Run id(s): {8988142}
[2018-04-11 21:23:09] Error in run 151: Run already exists in server. Run id(s): {8988137}
[2018-04-11 21:23:09] Started run 157 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:09] Started run 158 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:09] Started run 159 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:09] Started run 160 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:12] Error in run 153: Run already exists in server. Run id(s): {8988144}
[2018-04-11 21:23:12] Started run 161 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:13] Error in run 155: Run already exists in server. Run id(s): {8988148}
[2018-04-11 21:23:13] Started run 162 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:13] Error in run 154: Run already exists in server. Run id(s): {8988143}
[2018-04-11 21:23:13] Started run 163 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:13] Error in run 156: Run already exists in server. Run id(s): {8988149}
[2018-04-11 21:23:13] Started run 164 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:13] Error in run 160: Run already exists in server. Run id(s): {8988158}
[2018-04-11 21:23:13] Error in run 158: Run already exists in server. Run id(s): {8988151}
[2018-04-11 21:23:13] Error in run 159: Run already exists in server. Run id(s): {8988152}
[2018-04-11 21:23:13] Started run 165 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:13] Error in run 157: Run already exists in server. Run id(s): {8988150}
[2018-04-11 21:23:13] Started run 168 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:13] Started run 166 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:13] Started run 167 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:17] Error in run 161: Run already exists in server. Run id(s): {8988159}
[2018-04-11 21:23:17] Started run 169 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:17] Error in run 163: Run already exists in server. Run id(s): {8988161}
[2018-04-11 21:23:17] Error in run 162: Run already exists in server. Run id(s): {8988157}
[2018-04-11 21:23:17] Started run 170 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:17] Started run 171 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:17] Error in run 164: Run already exists in server. Run id(s): {8988162}
[2018-04-11 21:23:17] Started run 172 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:18] Error in run 168: Run already exists in server. Run id(s): {8988168}
[2018-04-11 21:23:18] Error in run 165: Run already exists in server. Run id(s): {8988166}
[2018-04-11 21:23:18] Started run 173 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:18] Started run 174 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:18] Error in run 166: Run already exists in server. Run id(s): {8988170}
[2018-04-11 21:23:18] Error in run 167: Run already exists in server. Run id(s): {8988169}
[2018-04-11 21:23:18] Started run 175 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:18] Started run 176 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:21] Error in run 169: Run already exists in server. Run id(s): {8988171}
[2018-04-11 21:23:21] Started run 177 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:22] Error in run 171: Run already exists in server. Run id(s): {8988174}
[2018-04-11 21:23:22] Error in run 170: Run already exists in server. Run id(s): {8988173}
[2018-04-11 21:23:22] Started run 178 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:22] Started run 179 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:22] Error in run 172: Run already exists in server. Run id(s): {8988185}
[2018-04-11 21:23:22] Started run 180 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:22] Error in run 173: Run already exists in server. Run id(s): {8988176}
[2018-04-11 21:23:22] Started run 181 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:22] Error in run 174: Run already exists in server. Run id(s): {8988178}
[2018-04-11 21:23:22] Started run 182 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:23] Error in run 176: Run already exists in server. Run id(s): {8988179}
[2018-04-11 21:23:23] Error in run 175: Run already exists in server. Run id(s): {8988180}
[2018-04-11 21:23:23] Started run 183 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:23] Started run 184 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:26] Error in run 177: Run already exists in server. Run id(s): {8988194}
[2018-04-11 21:23:26] Started run 185 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:26] Error in run 178: Run already exists in server. Run id(s): {8988186}
[2018-04-11 21:23:26] Error in run 179: Run already exists in server. Run id(s): {8988195}
[2018-04-11 21:23:26] Started run 186 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:26] Started run 187 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:27] Error in run 180: Run already exists in server. Run id(s): {8988193}
[2018-04-11 21:23:27] Started run 188 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:27] Error in run 181: Run already exists in server. Run id(s): {8988192}
[2018-04-11 21:23:27] Started run 189 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:27] Error in run 182: Run already exists in server. Run id(s): {8988196}
[2018-04-11 21:23:27] Started run 190 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:27] Error in run 183: Run already exists in server. Run id(s): {8988197}
[2018-04-11 21:23:27] Started run 191 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:27] Error in run 184: Run already exists in server. Run id(s): {8988199}
[2018-04-11 21:23:27] Started run 192 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:30] Error in run 185: Run already exists in server. Run id(s): {8988198}
[2018-04-11 21:23:30] Started run 193 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:31] Error in run 186: Run already exists in server. Run id(s): {8988210}
[2018-04-11 21:23:31] Started run 194 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:31] Error in run 187: Run already exists in server. Run id(s): {8988206}
[2018-04-11 21:23:31] Started run 195 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:31] Error in run 188: Run already exists in server. Run id(s): {8988200}
[2018-04-11 21:23:31] Started run 196 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:31] Error in run 189: Run already exists in server. Run id(s): {8988202}
[2018-04-11 21:23:31] Started run 197 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:32] Error in run 190: Run already exists in server. Run id(s): {8988207}
[2018-04-11 21:23:32] Started run 198 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:32] Error in run 191: Run already exists in server. Run id(s): {8988205}
[2018-04-11 21:23:32] Error in run 192: Run already exists in server. Run id(s): {8988209}
[2018-04-11 21:23:32] Started run 199 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:32] Started run 200 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:35] Error in run 193: Run already exists in server. Run id(s): {8988208}
[2018-04-11 21:23:35] Started run 201 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:35] Error in run 194: Run already exists in server. Run id(s): {8988211}
[2018-04-11 21:23:35] Started run 202 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:35] Error in run 195: Run already exists in server. Run id(s): {8988212}
[2018-04-11 21:23:35] Started run 203 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:36] Error in run 196: Run already exists in server. Run id(s): {8988213}
[2018-04-11 21:23:36] Started run 204 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:36] Error in run 197: Run already exists in server. Run id(s): {8988217}
[2018-04-11 21:23:36] Started run 205 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:36] Error in run 198: Run already exists in server. Run id(s): {8988215}
[2018-04-11 21:23:36] Started run 206 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:36] Error in run 200: Run already exists in server. Run id(s): {8988218}
[2018-04-11 21:23:36] Error in run 199: Run already exists in server. Run id(s): {8988228}
[2018-04-11 21:23:36] Started run 207 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:36] Started run 208 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:39] Error in run 201: Run already exists in server. Run id(s): {8988219}
[2018-04-11 21:23:39] Started run 209 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:40] Error in run 202: Run already exists in server. Run id(s): {8988223}
[2018-04-11 21:23:40] Started run 210 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:40] Error in run 204: Run already exists in server. Run id(s): {8988230}
[2018-04-11 21:23:40] Error in run 203: Run already exists in server. Run id(s): {8988221}
[2018-04-11 21:23:40] Started run 211 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:40] Started run 212 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:40] Error in run 205: Run already exists in server. Run id(s): {8988226}
[2018-04-11 21:23:40] Started run 213 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:41] Error in run 206: Run already exists in server. Run id(s): {8988227}
[2018-04-11 21:23:41] Error in run 208: Run already exists in server. Run id(s): {8988236}
[2018-04-11 21:23:41] Started run 214 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:41] Error in run 207: Run already exists in server. Run id(s): {8988229}
[2018-04-11 21:23:41] Started run 216 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:41] Started run 215 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:44] Error in run 209: Run already exists in server. Run id(s): {8988235}
[2018-04-11 21:23:44] Started run 217 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:44] Error in run 210: Run already exists in server. Run id(s): {8988238}
[2018-04-11 21:23:44] Started run 218 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:45] Error in run 211: Run already exists in server. Run id(s): {8988239}
[2018-04-11 21:23:45] Started run 219 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:45] Error in run 213: Run already exists in server. Run id(s): {8988243}
[2018-04-11 21:23:45] Started run 220 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:45] Error in run 212: Run already exists in server. Run id(s): {8988255}
[2018-04-11 21:23:45] Started run 221 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:46] Error in run 214: Run already exists in server. Run id(s): {8988249}
[2018-04-11 21:23:46] Started run 222 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:46] Error in run 216: Run already exists in server. Run id(s): {8988247}
[2018-04-11 21:23:46] Started run 223 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:46] Error in run 215: Run already exists in server. Run id(s): {8988248}
[2018-04-11 21:23:46] Started run 224 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:49] Error in run 217: Run already exists in server. Run id(s): {8988257}
[2018-04-11 21:23:49] Started run 225 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:49] Error in run 218: Run already exists in server. Run id(s): {8988259}
[2018-04-11 21:23:49] Started run 226 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:49] Error in run 219: Run already exists in server. Run id(s): {8988252}
[2018-04-11 21:23:49] Started run 227 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:50] Error in run 220: Run already exists in server. Run id(s): {8988265}
[2018-04-11 21:23:50] Error in run 221: Run already exists in server. Run id(s): {8988263}
[2018-04-11 21:23:50] Started run 228 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:50] Started run 229 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:50] Error in run 222: Run already exists in server. Run id(s): {8988262}
[2018-04-11 21:23:50] Started run 230 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:51] Error in run 223: Run already exists in server. Run id(s): {8988276}
[2018-04-11 21:23:51] Started run 231 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:51] Error in run 224: Run already exists in server. Run id(s): {8988266}
[2018-04-11 21:23:51] Started run 232 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:54] Error in run 225: Run already exists in server. Run id(s): {8988267}
[2018-04-11 21:23:54] Error in run 226: Run already exists in server. Run id(s): {8988270}
[2018-04-11 21:23:54] Started run 233 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:54] Started run 234 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:54] Error in run 227: Run already exists in server. Run id(s): {8988273}
[2018-04-11 21:23:54] Started run 235 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:54] Error in run 229: Run already exists in server. Run id(s): {8988275}
[2018-04-11 21:23:54] Started run 236 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:54] Error in run 228: Run already exists in server. Run id(s): {8988272}
[2018-04-11 21:23:54] Error in run 230: Run already exists in server. Run id(s): {8988277}
[2018-04-11 21:23:54] Started run 237 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:54] Started run 238 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:55] Error in run 232: Run already exists in server. Run id(s): {8988280}
[2018-04-11 21:23:55] Error in run 231: Run already exists in server. Run id(s): {8988278}
[2018-04-11 21:23:55] Started run 239 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:55] Started run 240 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:58] Error in run 233: Run already exists in server. Run id(s): {8988281}
[2018-04-11 21:23:58] Started run 241 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:58] Error in run 234: Run already exists in server. Run id(s): {8988283}
[2018-04-11 21:23:58] Started run 242 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:59] Error in run 235: Run already exists in server. Run id(s): {8988284}
[2018-04-11 21:23:59] Started run 243 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:59] Error in run 236: Run already exists in server. Run id(s): {8988303}
[2018-04-11 21:23:59] Started run 244 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:59] Error in run 238: Run already exists in server. Run id(s): {8988292}
[2018-04-11 21:23:59] Error in run 237: Run already exists in server. Run id(s): {8988285}
[2018-04-11 21:23:59] Started run 245 on task 9952, dataset 'phoneme'.
[2018-04-11 21:23:59] Started run 246 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:00] Error in run 239: Run already exists in server. Run id(s): {8988290}
[2018-04-11 21:24:00] Started run 247 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:00] Error in run 240: Run already exists in server. Run id(s): {8988291}
[2018-04-11 21:24:00] Started run 248 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:03] Error in run 241: Run already exists in server. Run id(s): {8988293}
[2018-04-11 21:24:03] Started run 249 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:03] Error in run 242: Run already exists in server. Run id(s): {8988298}
[2018-04-11 21:24:03] Started run 250 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:03] Error in run 243: Run already exists in server. Run id(s): {8988300}
[2018-04-11 21:24:03] Started run 251 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:03] Error in run 244: Run already exists in server. Run id(s): {8988304}
[2018-04-11 21:24:03] Started run 252 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:04] Error in run 246: Run already exists in server. Run id(s): {8988307}
[2018-04-11 21:24:04] Error in run 245: Run already exists in server. Run id(s): {8988305}
[2018-04-11 21:24:04] Started run 253 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:04] Started run 254 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:04] Error in run 247: Run already exists in server. Run id(s): {8988322}
[2018-04-11 21:24:04] Started run 255 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:05] Error in run 248: Run already exists in server. Run id(s): {8988314}
[2018-04-11 21:24:05] Started run 256 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:07] Error in run 249: Run already exists in server. Run id(s): {8988318}
[2018-04-11 21:24:07] Started run 257 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:08] Error in run 250: Run already exists in server. Run id(s): {8988321}
[2018-04-11 21:24:08] Started run 258 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:08] Error in run 251: Run already exists in server. Run id(s): {8988324}
[2018-04-11 21:24:08] Started run 259 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:08] Error in run 252: Run already exists in server. Run id(s): {8988327}
[2018-04-11 21:24:08] Started run 260 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:09] Error in run 253: Run already exists in server. Run id(s): {8988333}
[2018-04-11 21:24:09] Started run 261 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:09] Error in run 255: Run already exists in server. Run id(s): {8988330}
[2018-04-11 21:24:09] Started run 262 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:09] Error in run 254: Run already exists in server. Run id(s): {8988331}
[2018-04-11 21:24:09] Started run 263 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:09] Error in run 256: Run already exists in server. Run id(s): {8988343}
[2018-04-11 21:24:09] Started run 264 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:12] Error in run 257: Run already exists in server. Run id(s): {8988339}
[2018-04-11 21:24:12] Started run 265 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:12] Error in run 258: Run already exists in server. Run id(s): {8988346}
[2018-04-11 21:24:12] Started run 266 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:13] Error in run 259: Run already exists in server. Run id(s): {8988341}
[2018-04-11 21:24:13] Started run 267 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:13] Error in run 260: Run already exists in server. Run id(s): {8988347}
[2018-04-11 21:24:13] Started run 268 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:13] Error in run 261: Run already exists in server. Run id(s): {8988352}
[2018-04-11 21:24:13] Started run 269 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:13] Error in run 262: Run already exists in server. Run id(s): {8988349}
[2018-04-11 21:24:13] Error in run 263: Run already exists in server. Run id(s): {8988353}
[2018-04-11 21:24:13] Started run 270 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:13] Started run 271 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:14] Error in run 264: Run already exists in server. Run id(s): {8988355}
[2018-04-11 21:24:14] Started run 272 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:17] Error in run 265: Run already exists in server. Run id(s): {9014539, 8988357}
[2018-04-11 21:24:17] Error in run 266: Run already exists in server. Run id(s): {8988359}
[2018-04-11 21:24:17] Started run 273 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:17] Started run 274 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:17] Error in run 268: Run already exists in server. Run id(s): {8988363}
[2018-04-11 21:24:17] Started run 275 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:17] Error in run 267: Run already exists in server. Run id(s): {8988373}
[2018-04-11 21:24:17] Started run 276 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:18] Error in run 271: Run already exists in server. Run id(s): {8988367}
[2018-04-11 21:24:18] Error in run 269: Run already exists in server. Run id(s): {8988361}
[2018-04-11 21:24:18] Error in run 270: Run already exists in server. Run id(s): {8988368}
[2018-04-11 21:24:18] Started run 277 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:18] Started run 279 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:18] Started run 278 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:18] Error in run 272: Run already exists in server. Run id(s): {8988369}
[2018-04-11 21:24:18] Started run 280 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:21] Error in run 273: Run already exists in server. Run id(s): {8988372}
[2018-04-11 21:24:21] Started run 281 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:21] Error in run 274: Run already exists in server. Run id(s): {8988377}
[2018-04-11 21:24:21] Started run 282 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:22] Error in run 276: Run already exists in server. Run id(s): {8988395}
[2018-04-11 21:24:22] Error in run 275: Run already exists in server. Run id(s): {8988374}
[2018-04-11 21:24:22] Started run 283 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:22] Started run 284 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:23] Error in run 278: Run already exists in server. Run id(s): {8988379}
[2018-04-11 21:24:23] Started run 285 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:23] Error in run 277: Run already exists in server. Run id(s): {8988380}
[2018-04-11 21:24:23] Error in run 279: Run already exists in server. Run id(s): {8988388}
[2018-04-11 21:24:23] Started run 286 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:23] Started run 287 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:23] Error in run 280: Run already exists in server. Run id(s): {8988390}
[2018-04-11 21:24:23] Started run 288 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:26] Error in run 282: Run already exists in server. Run id(s): {8988396}
[2018-04-11 21:24:26] Started run 289 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:26] Error in run 281: Run already exists in server. Run id(s): {8988391}
[2018-04-11 21:24:26] Started run 290 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:27] Error in run 283: Run already exists in server. Run id(s): {8988399}
[2018-04-11 21:24:27] Started run 291 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:27] Error in run 284: Run already exists in server. Run id(s): {8988397}
[2018-04-11 21:24:27] Started run 292 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:27] Error in run 285: Run already exists in server. Run id(s): {8988409}
[2018-04-11 21:24:27] Started run 293 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:28] Error in run 288: Run already exists in server. Run id(s): {8988408}
[2018-04-11 21:24:28] Error in run 286: Run already exists in server. Run id(s): {8988404}
[2018-04-11 21:24:28] Started run 294 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:28] Error in run 287: Run already exists in server. Run id(s): {8988406}
[2018-04-11 21:24:28] Started run 295 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:28] Started run 296 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:31] Error in run 290: Run already exists in server. Run id(s): {8988421}
[2018-04-11 21:24:31] Error in run 289: Run already exists in server. Run id(s): {8988413}
[2018-04-11 21:24:31] Started run 297 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:31] Started run 298 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:31] Error in run 291: Run already exists in server. Run id(s): {8988416}
[2018-04-11 21:24:31] Started run 299 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:31] Error in run 292: Run already exists in server. Run id(s): {8988417}
[2018-04-11 21:24:31] Started run 300 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:32] Error in run 293: Run already exists in server. Run id(s): {8988422}
[2018-04-11 21:24:32] Started run 301 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:32] Error in run 294: Run already exists in server. Run id(s): {8988429}
[2018-04-11 21:24:32] Error in run 295: Run already exists in server. Run id(s): {8988432}
[2018-04-11 21:24:32] Started run 302 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:32] Started run 303 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:32] Error in run 296: Run already exists in server. Run id(s): {8988447}
[2018-04-11 21:24:32] Started run 304 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:35] Error in run 298: Run already exists in server. Run id(s): {8988435}
[2018-04-11 21:24:35] Started run 305 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:35] Error in run 297: Run already exists in server. Run id(s): {8988436}
[2018-04-11 21:24:35] Started run 306 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:36] Error in run 299: Run already exists in server. Run id(s): {8988441}
[2018-04-11 21:24:36] Started run 307 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:36] Error in run 300: Run already exists in server. Run id(s): {8988450}
[2018-04-11 21:24:36] Started run 308 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:36] Error in run 301: Run already exists in server. Run id(s): {8988442}
[2018-04-11 21:24:36] Started run 309 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:37] Error in run 302: Run already exists in server. Run id(s): {8988448}
[2018-04-11 21:24:37] Started run 310 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:37] Error in run 303: Run already exists in server. Run id(s): {8988449}
[2018-04-11 21:24:37] Started run 311 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:37] Error in run 304: Run already exists in server. Run id(s): {8988465}
[2018-04-11 21:24:37] Started run 312 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:40] Error in run 306: Run already exists in server. Run id(s): {8988456}
[2018-04-11 21:24:40] Error in run 305: Run already exists in server. Run id(s): {8988454}
[2018-04-11 21:24:40] Started run 313 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:40] Started run 314 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:40] Error in run 307: Run already exists in server. Run id(s): {8988457}
[2018-04-11 21:24:40] Started run 315 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:40] Error in run 308: Run already exists in server. Run id(s): {8988458}
[2018-04-11 21:24:40] Started run 316 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:41] Error in run 309: Run already exists in server. Run id(s): {8988461}
[2018-04-11 21:24:41] Started run 317 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:41] Error in run 310: Run already exists in server. Run id(s): {8988469}
[2018-04-11 21:24:41] Started run 318 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:41] Error in run 311: Run already exists in server. Run id(s): {8988468}
[2018-04-11 21:24:41] Started run 319 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:41] Error in run 312: Run already exists in server. Run id(s): {8988470}
[2018-04-11 21:24:41] Started run 320 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:44] Error in run 314: Run already exists in server. Run id(s): {8988481}
[2018-04-11 21:24:44] Error in run 313: Run already exists in server. Run id(s): {8988480}
[2018-04-11 21:24:44] Started run 321 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:44] Started run 322 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:45] Error in run 315: Run already exists in server. Run id(s): {8988479}
[2018-04-11 21:24:45] Error in run 316: Run already exists in server. Run id(s): {8988478}
[2018-04-11 21:24:45] Started run 323 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:45] Started run 324 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:46] Error in run 317: Run already exists in server. Run id(s): {8988488}
[2018-04-11 21:24:46] Started run 325 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:46] Error in run 318: Run already exists in server. Run id(s): {8988486}
[2018-04-11 21:24:46] Started run 326 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:46] Error in run 319: Run already exists in server. Run id(s): {8988487}
[2018-04-11 21:24:46] Started run 327 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:46] Error in run 320: Run already exists in server. Run id(s): {8988495}
[2018-04-11 21:24:46] Started run 328 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:49] Error in run 321: Run already exists in server. Run id(s): {8988500}
[2018-04-11 21:24:49] Error in run 322: Run already exists in server. Run id(s): {8988509}
[2018-04-11 21:24:49] Started run 329 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:49] Started run 330 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:50] Error in run 323: Run already exists in server. Run id(s): {8988494}
[2018-04-11 21:24:50] Error in run 324: Run already exists in server. Run id(s): {8988503}
[2018-04-11 21:24:50] Started run 331 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:50] Started run 332 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:51] Error in run 327: Run already exists in server. Run id(s): {8988506}
[2018-04-11 21:24:51] Started run 333 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:51] Error in run 328: Run already exists in server. Run id(s): {8988508}
[2018-04-11 21:24:51] Error in run 325: Run already exists in server. Run id(s): {8988504}
[2018-04-11 21:24:51] Started run 334 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:51] Error in run 326: Run already exists in server. Run id(s): {8988505}
[2018-04-11 21:24:51] Started run 336 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:51] Started run 335 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:54] Error in run 330: Run already exists in server. Run id(s): {8988522}
[2018-04-11 21:24:54] Error in run 329: Run already exists in server. Run id(s): {8988507}
[2018-04-11 21:24:54] Started run 337 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:54] Started run 338 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:55] Error in run 332: Run already exists in server. Run id(s): {8988513}
[2018-04-11 21:24:55] Error in run 331: Run already exists in server. Run id(s): {8988511}
[2018-04-11 21:24:55] Started run 339 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:55] Started run 340 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:55] Error in run 333: Run already exists in server. Run id(s): {8988512}
[2018-04-11 21:24:55] Started run 341 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:55] Error in run 335: Run already exists in server. Run id(s): {8988516}
[2018-04-11 21:24:55] Error in run 334: Run already exists in server. Run id(s): {8988514}
[2018-04-11 21:24:55] Error in run 336: Run already exists in server. Run id(s): {8988520}
[2018-04-11 21:24:55] Started run 342 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:55] Started run 343 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:55] Started run 344 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:58] Error in run 337: Run already exists in server. Run id(s): {8988521}
[2018-04-11 21:24:58] Started run 345 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:58] Error in run 338: Run already exists in server. Run id(s): {8988523}
[2018-04-11 21:24:58] Started run 346 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:59] Error in run 339: Run already exists in server. Run id(s): {8988527}
[2018-04-11 21:24:59] Started run 347 on task 9952, dataset 'phoneme'.
[2018-04-11 21:24:59] Error in run 340: Run already exists in server. Run id(s): {8988528}
[2018-04-11 21:24:59] Started run 348 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:00] Error in run 341: Run already exists in server. Run id(s): {8988530}
[2018-04-11 21:25:00] Started run 349 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:00] Error in run 342: Run already exists in server. Run id(s): {8988529}
[2018-04-11 21:25:00] Started run 350 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:00] Error in run 344: Run already exists in server. Run id(s): {8988540}
[2018-04-11 21:25:00] Started run 351 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:00] Error in run 343: Run already exists in server. Run id(s): {8988537}
[2018-04-11 21:25:00] Started run 352 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:03] Error in run 345: Run already exists in server. Run id(s): {8988535}
[2018-04-11 21:25:03] Error in run 346: Run already exists in server. Run id(s): {8988536}
[2018-04-11 21:25:03] Started run 353 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:03] Started run 354 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:04] Error in run 348: Run already exists in server. Run id(s): {8988552}
[2018-04-11 21:25:04] Error in run 347: Run already exists in server. Run id(s): {8988541}
[2018-04-11 21:25:04] Started run 355 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:04] Started run 356 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:04] Error in run 349: Run already exists in server. Run id(s): {8988551}
[2018-04-11 21:25:04] Started run 357 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:04] Error in run 351: Run already exists in server. Run id(s): {8988547}
[2018-04-11 21:25:04] Started run 358 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:04] Error in run 350: Run already exists in server. Run id(s): {8988545}
[2018-04-11 21:25:04] Started run 359 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:05] Error in run 352: Run already exists in server. Run id(s): {8988548}
[2018-04-11 21:25:05] Started run 360 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:07] Error in run 353: Run already exists in server. Run id(s): {8988550}
[2018-04-11 21:25:07] Error in run 354: Run already exists in server. Run id(s): {8988553}
[2018-04-11 21:25:07] Started run 361 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:07] Started run 362 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:08] Error in run 355: Run already exists in server. Run id(s): {8988556}
[2018-04-11 21:25:08] Started run 363 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:08] Error in run 356: Run already exists in server. Run id(s): {8988566}
[2018-04-11 21:25:08] Started run 364 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:09] Error in run 357: Run already exists in server. Run id(s): {8988557}
[2018-04-11 21:25:09] Started run 365 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:09] Error in run 359: Run already exists in server. Run id(s): {8988576}
[2018-04-11 21:25:09] Started run 366 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:09] Error in run 358: Run already exists in server. Run id(s): {8988567}
[2018-04-11 21:25:09] Started run 367 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:09] Error in run 360: Run already exists in server. Run id(s): {8988562}
[2018-04-11 21:25:09] Started run 368 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:12] Error in run 362: Run already exists in server. Run id(s): {8988573}
[2018-04-11 21:25:12] Error in run 361: Run already exists in server. Run id(s): {8988569}
[2018-04-11 21:25:12] Started run 369 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:12] Started run 370 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:13] Error in run 364: Run already exists in server. Run id(s): {8988574}
[2018-04-11 21:25:13] Error in run 363: Run already exists in server. Run id(s): {8988575}
[2018-04-11 21:25:13] Started run 371 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:13] Started run 372 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:14] Error in run 366: Run already exists in server. Run id(s): {8988578}
[2018-04-11 21:25:14] Started run 373 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:14] Error in run 367: Run already exists in server. Run id(s): {8988580}
[2018-04-11 21:25:14] Error in run 365: Run already exists in server. Run id(s): {8988577}
[2018-04-11 21:25:14] Started run 374 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:14] Started run 375 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:14] Error in run 368: Run already exists in server. Run id(s): {8988583}
[2018-04-11 21:25:14] Started run 376 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:17] Error in run 369: Run already exists in server. Run id(s): {8988590}
[2018-04-11 21:25:17] Error in run 370: Run already exists in server. Run id(s): {8988593}
[2018-04-11 21:25:17] Started run 377 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:17] Started run 378 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:17] Error in run 371: Run already exists in server. Run id(s): {8988592}
[2018-04-11 21:25:17] Started run 379 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:17] Error in run 372: Run already exists in server. Run id(s): {8988589}
[2018-04-11 21:25:17] Started run 380 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:18] Error in run 373: Run already exists in server. Run id(s): {8988600, 9014816}
[2018-04-11 21:25:18] Error in run 376: Run already exists in server. Run id(s): {8988604}
[2018-04-11 21:25:18] Started run 381 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:18] Error in run 374: Run already exists in server. Run id(s): {8988599}
[2018-04-11 21:25:18] Error in run 375: Run already exists in server. Run id(s): {8988603}
[2018-04-11 21:25:18] Started run 382 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:18] Started run 383 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:18] Started run 384 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:22] Error in run 378: Run already exists in server. Run id(s): {8988607}
[2018-04-11 21:25:22] Error in run 377: Run already exists in server. Run id(s): {8988606}
[2018-04-11 21:25:22] Started run 385 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:22] Started run 386 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:22] Error in run 379: Run already exists in server. Run id(s): {8988613}
[2018-04-11 21:25:22] Started run 387 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:22] Error in run 380: Run already exists in server. Run id(s): {8988608}
[2018-04-11 21:25:22] Started run 388 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:23] Error in run 384: Run already exists in server. Run id(s): {8988617}
[2018-04-11 21:25:23] Error in run 383: Run already exists in server. Run id(s): {8988612}
[2018-04-11 21:25:23] Error in run 381: Run already exists in server. Run id(s): {8988609}
[2018-04-11 21:25:23] Error in run 382: Run already exists in server. Run id(s): {8988610}
[2018-04-11 21:25:23] Started run 389 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:23] Started run 391 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:23] Started run 390 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:23] Started run 392 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:26] Error in run 385: Run already exists in server. Run id(s): {8988620}
[2018-04-11 21:25:26] Started run 393 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:26] Error in run 386: Run already exists in server. Run id(s): {8988630}
[2018-04-11 21:25:26] Started run 394 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:27] Error in run 387: Run already exists in server. Run id(s): {8988621}
[2018-04-11 21:25:27] Started run 395 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:27] Error in run 388: Run already exists in server. Run id(s): {8988623}
[2018-04-11 21:25:27] Started run 396 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:28] Error in run 389: Run already exists in server. Run id(s): {8988639}
[2018-04-11 21:25:28] Started run 397 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:28] Error in run 391: Run already exists in server. Run id(s): {8988631}
[2018-04-11 21:25:28] Started run 398 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:28] Error in run 390: Run already exists in server. Run id(s): {8988633}
[2018-04-11 21:25:28] Started run 399 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:28] Error in run 392: Run already exists in server. Run id(s): {8988636}
[2018-04-11 21:25:28] Started run 400 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:31] Error in run 393: Run already exists in server. Run id(s): {8988638}
[2018-04-11 21:25:31] Started run 401 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:31] Error in run 394: Run already exists in server. Run id(s): {8988637}
[2018-04-11 21:25:31] Started run 402 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:31] Error in run 395: Run already exists in server. Run id(s): {8988640}
[2018-04-11 21:25:31] Started run 403 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:32] Error in run 396: Run already exists in server. Run id(s): {8988656}
[2018-04-11 21:25:32] Started run 404 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:32] Error in run 397: Run already exists in server. Run id(s): {8988642}
[2018-04-11 21:25:32] Started run 405 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:32] Error in run 398: Run already exists in server. Run id(s): {8988644}
[2018-04-11 21:25:32] Started run 406 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:33] Error in run 400: Run already exists in server. Run id(s): {8988657}
[2018-04-11 21:25:33] Started run 407 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:33] Error in run 399: Run already exists in server. Run id(s): {8988643}
[2018-04-11 21:25:33] Started run 408 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:35] Error in run 401: Run already exists in server. Run id(s): {8988651}
[2018-04-11 21:25:35] Started run 409 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:35] Error in run 402: Run already exists in server. Run id(s): {8988649}
[2018-04-11 21:25:35] Started run 410 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:36] Error in run 403: Run already exists in server. Run id(s): {8988655}
[2018-04-11 21:25:36] Started run 411 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:36] Error in run 404: Run already exists in server. Run id(s): {8988660}
[2018-04-11 21:25:36] Started run 412 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:37] Error in run 405: Run already exists in server. Run id(s): {8988666}
[2018-04-11 21:25:37] Started run 413 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:37] Error in run 406: Run already exists in server. Run id(s): {8988665}
[2018-04-11 21:25:37] Started run 414 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:37] Error in run 407: Run already exists in server. Run id(s): {8988667}
[2018-04-11 21:25:37] Error in run 408: Run already exists in server. Run id(s): {8988668}
[2018-04-11 21:25:37] Started run 415 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:37] Started run 416 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:40] Error in run 409: Run already exists in server. Run id(s): {8988671}
[2018-04-11 21:25:40] Started run 417 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:40] Error in run 410: Run already exists in server. Run id(s): {8988670}
[2018-04-11 21:25:40] Started run 418 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:41] Error in run 411: Run already exists in server. Run id(s): {8988674}
[2018-04-11 21:25:41] Started run 419 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:41] Error in run 412: Run already exists in server. Run id(s): {8988678}
[2018-04-11 21:25:41] Started run 420 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:41] Error in run 414: Run already exists in server. Run id(s): {8988675}
[2018-04-11 21:25:41] Started run 421 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:41] Error in run 413: Run already exists in server. Run id(s): {8988673}
[2018-04-11 21:25:41] Started run 422 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:42] Error in run 415: Run already exists in server. Run id(s): {8988676}
[2018-04-11 21:25:42] Error in run 416: Run already exists in server. Run id(s): {8988679}
[2018-04-11 21:25:42] Started run 423 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:42] Started run 424 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:44] Error in run 418: Run already exists in server. Run id(s): {8988696}
[2018-04-11 21:25:44] Error in run 417: Run already exists in server. Run id(s): {8988682}
[2018-04-11 21:25:44] Started run 425 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:44] Started run 426 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:45] Error in run 419: Run already exists in server. Run id(s): {8988684}
[2018-04-11 21:25:45] Started run 427 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:45] Error in run 420: Run already exists in server. Run id(s): {8988687}
[2018-04-11 21:25:45] Started run 428 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:46] Error in run 421: Run already exists in server. Run id(s): {8988688}
[2018-04-11 21:25:46] Started run 429 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:46] Error in run 422: Run already exists in server. Run id(s): {8988689}
[2018-04-11 21:25:46] Started run 430 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:46] Error in run 424: Run already exists in server. Run id(s): {8988692}
[2018-04-11 21:25:46] Error in run 423: Run already exists in server. Run id(s): {8988706}
[2018-04-11 21:25:46] Started run 431 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:46] Started run 432 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:49] Error in run 425: Run already exists in server. Run id(s): {8988709}
[2018-04-11 21:25:49] Error in run 426: Run already exists in server. Run id(s): {8988704}
[2018-04-11 21:25:49] Started run 433 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:49] Started run 434 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:50] Error in run 427: Run already exists in server. Run id(s): {8988705}
[2018-04-11 21:25:50] Started run 435 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:50] Error in run 428: Run already exists in server. Run id(s): {8988710}
[2018-04-11 21:25:50] Started run 436 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:50] Error in run 429: Run already exists in server. Run id(s): {8988711}
[2018-04-11 21:25:50] Started run 437 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:50] Error in run 430: Run already exists in server. Run id(s): {8988712}
[2018-04-11 21:25:50] Started run 438 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:51] Error in run 432: Run already exists in server. Run id(s): {8988720}
[2018-04-11 21:25:51] Error in run 431: Run already exists in server. Run id(s): {8988714}
[2018-04-11 21:25:51] Started run 439 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:51] Started run 440 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:53] Error in run 433: Run already exists in server. Run id(s): {8988716}
[2018-04-11 21:25:53] Started run 441 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:53] Error in run 434: Run already exists in server. Run id(s): {8988717}
[2018-04-11 21:25:53] Started run 442 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:54] Error in run 435: Run already exists in server. Run id(s): {8988719}
[2018-04-11 21:25:54] Started run 443 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:54] Error in run 436: Run already exists in server. Run id(s): {8988722}
[2018-04-11 21:25:54] Started run 444 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:55] Error in run 437: Run already exists in server. Run id(s): {8988723}
[2018-04-11 21:25:55] Started run 445 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:55] Error in run 438: Run already exists in server. Run id(s): {8988727}
[2018-04-11 21:25:55] Started run 446 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:55] Error in run 439: Run already exists in server. Run id(s): {8988740}
[2018-04-11 21:25:55] Error in run 440: Run already exists in server. Run id(s): {8988730}
[2018-04-11 21:25:55] Started run 447 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:55] Started run 448 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:58] Error in run 441: Run already exists in server. Run id(s): {8988735}
[2018-04-11 21:25:58] Error in run 442: Run already exists in server. Run id(s): {8988737}
[2018-04-11 21:25:58] Started run 449 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:58] Started run 450 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:59] Error in run 443: Run already exists in server. Run id(s): {8988741}
[2018-04-11 21:25:59] Started run 451 on task 9952, dataset 'phoneme'.
[2018-04-11 21:25:59] Error in run 444: Run already exists in server. Run id(s): {8988743}
[2018-04-11 21:25:59] Started run 452 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:00] Error in run 445: Run already exists in server. Run id(s): {8988744}
[2018-04-11 21:26:00] Error in run 446: Run already exists in server. Run id(s): {8988752}
[2018-04-11 21:26:00] Started run 453 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:00] Started run 454 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:00] Error in run 447: Run already exists in server. Run id(s): {8988748}
[2018-04-11 21:26:00] Started run 455 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:00] Error in run 448: Run already exists in server. Run id(s): {8988750}
[2018-04-11 21:26:00] Started run 456 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:03] Error in run 449: Run already exists in server. Run id(s): {8988751}
[2018-04-11 21:26:03] Started run 457 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:03] Error in run 450: Run already exists in server. Run id(s): {8988759}
[2018-04-11 21:26:03] Started run 458 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:03] Error in run 451: Run already exists in server. Run id(s): {8988758}
[2018-04-11 21:26:03] Started run 459 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:03] Error in run 452: Run already exists in server. Run id(s): {8988757}
[2018-04-11 21:26:03] Started run 460 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:04] Error in run 454: Run already exists in server. Run id(s): {8988763}
[2018-04-11 21:26:04] Error in run 453: Run already exists in server. Run id(s): {8988773}
[2018-04-11 21:26:04] Started run 461 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:04] Started run 462 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:04] Error in run 455: Run already exists in server. Run id(s): {8988764}
[2018-04-11 21:26:04] Error in run 456: Run already exists in server. Run id(s): {8988765}
[2018-04-11 21:26:04] Started run 463 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:04] Started run 464 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:07] Error in run 457: Run already exists in server. Run id(s): {8988767}
[2018-04-11 21:26:07] Started run 465 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:07] Error in run 458: Run already exists in server. Run id(s): {8988772}
[2018-04-11 21:26:07] Started run 466 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:08] Error in run 459: Run already exists in server. Run id(s): {8988770}
[2018-04-11 21:26:08] Started run 467 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:08] Error in run 460: Run already exists in server. Run id(s): {8988781}
[2018-04-11 21:26:08] Started run 468 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:09] Error in run 461: Run already exists in server. Run id(s): {8988786}
[2018-04-11 21:26:09] Error in run 462: Run already exists in server. Run id(s): {9015304, 8988791}
[2018-04-11 21:26:09] Started run 469 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:09] Started run 470 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:09] Error in run 463: Run already exists in server. Run id(s): {8988782}
[2018-04-11 21:26:09] Started run 471 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:09] Error in run 464: Run already exists in server. Run id(s): {8988789}
[2018-04-11 21:26:09] Started run 472 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:12] Error in run 465: Run already exists in server. Run id(s): {8988788}
[2018-04-11 21:26:12] Started run 473 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:12] Error in run 466: Run already exists in server. Run id(s): {8988790}
[2018-04-11 21:26:12] Started run 474 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:12] Error in run 467: Run already exists in server. Run id(s): {8988794}
[2018-04-11 21:26:12] Started run 475 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:13] Error in run 468: Run already exists in server. Run id(s): {8988801}
[2018-04-11 21:26:13] Started run 476 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:13] Error in run 469: Run already exists in server. Run id(s): {8988800}
[2018-04-11 21:26:13] Error in run 470: Run already exists in server. Run id(s): {8988799}
[2018-04-11 21:26:13] Started run 477 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:13] Started run 478 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:13] Error in run 471: Run already exists in server. Run id(s): {8988803}
[2018-04-11 21:26:13] Started run 479 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:14] Error in run 472: Run already exists in server. Run id(s): {8988808}
[2018-04-11 21:26:14] Started run 480 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:16] Error in run 473: Run already exists in server. Run id(s): {8988806}
[2018-04-11 21:26:16] Started run 481 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:17] Error in run 474: Run already exists in server. Run id(s): {8988809}
[2018-04-11 21:26:17] Started run 482 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:17] Error in run 475: Run already exists in server. Run id(s): {8988819}
[2018-04-11 21:26:17] Started run 483 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:18] Error in run 477: Run already exists in server. Run id(s): {8988812}
[2018-04-11 21:26:18] Started run 484 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:18] Error in run 478: Run already exists in server. Run id(s): {8988815}
[2018-04-11 21:26:18] Started run 485 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:18] Error in run 476: Run already exists in server. Run id(s): {8988810}
[2018-04-11 21:26:18] Started run 486 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:18] Error in run 479: Run already exists in server. Run id(s): {8988817}
[2018-04-11 21:26:18] Started run 487 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:18] Error in run 480: Run already exists in server. Run id(s): {8988820}
[2018-04-11 21:26:18] Started run 488 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:21] Error in run 481: Run already exists in server. Run id(s): {8988822}
[2018-04-11 21:26:21] Started run 489 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:21] Error in run 482: Run already exists in server. Run id(s): {8988821}
[2018-04-11 21:26:21] Started run 490 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:22] Error in run 483: Run already exists in server. Run id(s): {8988833}
[2018-04-11 21:26:22] Started run 491 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:22] Error in run 487: Run already exists in server. Run id(s): {8988836}
[2018-04-11 21:26:22] Error in run 485: Run already exists in server. Run id(s): {8988828}
[2018-04-11 21:26:22] Error in run 486: Run already exists in server. Run id(s): {8988831}
[2018-04-11 21:26:22] Error in run 484: Run already exists in server. Run id(s): {8988829}
[2018-04-11 21:26:22] Started run 492 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:22] Started run 493 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:22] Started run 494 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:22] Started run 495 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:23] Error in run 488: Run already exists in server. Run id(s): {8988835}
[2018-04-11 21:26:23] Started run 496 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:26] Error in run 489: Run already exists in server. Run id(s): {8988837}
[2018-04-11 21:26:26] Started run 497 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:26] Error in run 490: Run already exists in server. Run id(s): {8988838}
[2018-04-11 21:26:26] Started run 498 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:26] Error in run 491: Run already exists in server. Run id(s): {8988853}
[2018-04-11 21:26:26] Started run 499 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:27] Error in run 492: Run already exists in server. Run id(s): {8988843}
[2018-04-11 21:26:27] Started run 500 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:27] Error in run 493: Run already exists in server. Run id(s): {8988844}
[2018-04-11 21:26:27] Started run 501 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:27] Error in run 494: Run already exists in server. Run id(s): {8988845}
[2018-04-11 21:26:27] Error in run 495: Run already exists in server. Run id(s): {8988852}
[2018-04-11 21:26:27] Started run 502 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:27] Started run 503 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:27] Error in run 496: Run already exists in server. Run id(s): {8988860}
[2018-04-11 21:26:27] Started run 504 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:30] Error in run 498: Run already exists in server. Run id(s): {8988854}
[2018-04-11 21:26:30] Error in run 497: Run already exists in server. Run id(s): {8988850}
[2018-04-11 21:26:30] Started run 505 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:30] Started run 506 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:30] Error in run 499: Run already exists in server. Run id(s): {8988862}
[2018-04-11 21:26:30] Started run 507 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:32] Error in run 501: Run already exists in server. Run id(s): {8988861}
[2018-04-11 21:26:32] Error in run 503: Run already exists in server. Run id(s): {8988865}
[2018-04-11 21:26:32] Error in run 500: Run already exists in server. Run id(s): {8988859}
[2018-04-11 21:26:32] Error in run 502: Run already exists in server. Run id(s): {8988867}
[2018-04-11 21:26:32] Started run 508 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:32] Error in run 504: Run already exists in server. Run id(s): {8988866}
[2018-04-11 21:26:32] Started run 512 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:32] Started run 511 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:32] Started run 509 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:32] Started run 510 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:35] Error in run 505: Run already exists in server. Run id(s): {8988868}
[2018-04-11 21:26:35] Started run 513 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:35] Error in run 506: Run already exists in server. Run id(s): {8988871}
[2018-04-11 21:26:35] Started run 514 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:35] Error in run 507: Run already exists in server. Run id(s): {8988872}
[2018-04-11 21:26:35] Started run 515 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:36] Error in run 508: Run already exists in server. Run id(s): {8988873}
[2018-04-11 21:26:36] Started run 516 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:36] Error in run 509: Run already exists in server. Run id(s): {8988874}
[2018-04-11 21:26:36] Started run 517 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:36] Error in run 510: Run already exists in server. Run id(s): {8988879}
[2018-04-11 21:26:36] Error in run 511: Run already exists in server. Run id(s): {8988880}
[2018-04-11 21:26:36] Error in run 512: Run already exists in server. Run id(s): {8988881}
[2018-04-11 21:26:36] Started run 518 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:36] Started run 520 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:36] Started run 519 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:39] Error in run 513: Run already exists in server. Run id(s): {8988882}
[2018-04-11 21:26:39] Started run 521 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:40] Error in run 514: Run already exists in server. Run id(s): {8988885}
[2018-04-11 21:26:40] Error in run 515: Run already exists in server. Run id(s): {8988886}
[2018-04-11 21:26:40] Started run 522 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:40] Started run 523 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:41] Error in run 516: Run already exists in server. Run id(s): {8988887}
[2018-04-11 21:26:41] Started run 524 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:41] Error in run 520: Run already exists in server. Run id(s): {8988892}
[2018-04-11 21:26:41] Error in run 517: Run already exists in server. Run id(s): {8988888}
[2018-04-11 21:26:41] Started run 525 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:41] Error in run 518: Run already exists in server. Run id(s): {8988895}
[2018-04-11 21:26:41] Error in run 519: Run already exists in server. Run id(s): {8988894}
[2018-04-11 21:26:41] Started run 526 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:41] Started run 527 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:41] Started run 528 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:44] Error in run 521: Run already exists in server. Run id(s): {8988893}
[2018-04-11 21:26:44] Started run 529 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:44] Error in run 522: Run already exists in server. Run id(s): {8988897}
[2018-04-11 21:26:44] Error in run 523: Run already exists in server. Run id(s): {8988904}
[2018-04-11 21:26:44] Started run 531 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:44] Started run 530 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:45] Error in run 524: Run already exists in server. Run id(s): {8988898}
[2018-04-11 21:26:45] Started run 532 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:45] Error in run 527: Run already exists in server. Run id(s): {8988906}
[2018-04-11 21:26:45] Error in run 528: Run already exists in server. Run id(s): {8988909}
[2018-04-11 21:26:45] Started run 533 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:45] Error in run 526: Run already exists in server. Run id(s): {8988901}
[2018-04-11 21:26:45] Error in run 525: Run already exists in server. Run id(s): {8988899}
[2018-04-11 21:26:45] Started run 535 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:45] Started run 536 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:45] Started run 534 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:49] Error in run 530: Run already exists in server. Run id(s): {8988915}
[2018-04-11 21:26:49] Error in run 531: Run already exists in server. Run id(s): {8988922}
[2018-04-11 21:26:49] Error in run 529: Run already exists in server. Run id(s): {8988907}
[2018-04-11 21:26:49] Started run 537 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:49] Started run 538 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:49] Started run 539 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:50] Error in run 532: Run already exists in server. Run id(s): {8988917}
[2018-04-11 21:26:50] Started run 540 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:50] Error in run 533: Run already exists in server. Run id(s): {8988921}
[2018-04-11 21:26:50] Started run 541 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:50] Error in run 535: Run already exists in server. Run id(s): {8988925}
[2018-04-11 21:26:50] Started run 542 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:50] Error in run 536: Run already exists in server. Run id(s): {8988927}
[2018-04-11 21:26:50] Error in run 534: Run already exists in server. Run id(s): {8988933}
[2018-04-11 21:26:50] Started run 543 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:50] Started run 544 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:53] Error in run 537: Run already exists in server. Run id(s): {8988931}
[2018-04-11 21:26:53] Started run 545 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:53] Error in run 539: Run already exists in server. Run id(s): {8988929}
[2018-04-11 21:26:53] Error in run 538: Run already exists in server. Run id(s): {8988932}
[2018-04-11 21:26:53] Started run 546 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:53] Started run 547 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:54] Error in run 540: Run already exists in server. Run id(s): {8988939}
[2018-04-11 21:26:54] Started run 548 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:54] Error in run 541: Run already exists in server. Run id(s): {8988936}
[2018-04-11 21:26:54] Started run 549 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:54] Error in run 542: Run already exists in server. Run id(s): {8988937}
[2018-04-11 21:26:54] Started run 550 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:55] Error in run 544: Run already exists in server. Run id(s): {8988946}
[2018-04-11 21:26:55] Error in run 543: Run already exists in server. Run id(s): {8988938}
[2018-04-11 21:26:55] Started run 551 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:55] Started run 552 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:58] Error in run 545: Run already exists in server. Run id(s): {8988957}
[2018-04-11 21:26:58] Started run 553 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:58] Error in run 547: Run already exists in server. Run id(s): {8988949}
[2018-04-11 21:26:58] Error in run 546: Run already exists in server. Run id(s): {8988945, 9015507}
[2018-04-11 21:26:58] Started run 554 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:58] Started run 555 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:59] Error in run 549: Run already exists in server. Run id(s): {8988954}
[2018-04-11 21:26:59] Started run 556 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:59] Error in run 550: Run already exists in server. Run id(s): {8988953}
[2018-04-11 21:26:59] Error in run 548: Run already exists in server. Run id(s): {8988950}
[2018-04-11 21:26:59] Started run 557 on task 9952, dataset 'phoneme'.
[2018-04-11 21:26:59] Started run 558 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:00] Error in run 552: Run already exists in server. Run id(s): {8988976}
[2018-04-11 21:27:00] Started run 559 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:00] Error in run 551: Run already exists in server. Run id(s): {8988969}
[2018-04-11 21:27:00] Started run 560 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:02] Error in run 553: Run already exists in server. Run id(s): {8988960, 9015575}
[2018-04-11 21:27:02] Started run 561 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:03] Error in run 554: Run already exists in server. Run id(s): {8988962}
[2018-04-11 21:27:03] Started run 562 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:03] Error in run 555: Run already exists in server. Run id(s): {8988983}
[2018-04-11 21:27:03] Started run 563 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:04] Error in run 556: Run already exists in server. Run id(s): {8988965}
[2018-04-11 21:27:04] Started run 564 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:04] Error in run 558: Run already exists in server. Run id(s): {8988981}
[2018-04-11 21:27:04] Started run 565 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:04] Error in run 557: Run already exists in server. Run id(s): {8988987}
[2018-04-11 21:27:04] Started run 566 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:04] Error in run 559: Run already exists in server. Run id(s): {8988995}
[2018-04-11 21:27:04] Error in run 560: Run already exists in server. Run id(s): {8988986}
[2018-04-11 21:27:04] Started run 567 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:04] Started run 568 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:07] Error in run 561: Run already exists in server. Run id(s): {8988990}
[2018-04-11 21:27:07] Started run 569 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:07] Error in run 562: Run already exists in server. Run id(s): {8988994}
[2018-04-11 21:27:07] Started run 570 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:08] Error in run 563: Run already exists in server. Run id(s): {8988997}
[2018-04-11 21:27:08] Started run 571 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:08] Error in run 566: Run already exists in server. Run id(s): {8988998}
[2018-04-11 21:27:08] Started run 572 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:08] Error in run 564: Run already exists in server. Run id(s): {8988996}
[2018-04-11 21:27:08] Started run 573 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:08] Error in run 565: Run already exists in server. Run id(s): {8988999}
[2018-04-11 21:27:08] Started run 574 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:09] Error in run 568: Run already exists in server. Run id(s): {8989003}
[2018-04-11 21:27:09] Error in run 567: Run already exists in server. Run id(s): {8989001}
[2018-04-11 21:27:09] Started run 575 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:09] Started run 576 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:11] Error in run 569: Run already exists in server. Run id(s): {8989015}
[2018-04-11 21:27:11] Started run 577 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:12] Error in run 570: Run already exists in server. Run id(s): {8989010}
[2018-04-11 21:27:12] Started run 578 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:12] Error in run 571: Run already exists in server. Run id(s): {8989012}
[2018-04-11 21:27:12] Started run 579 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:13] Error in run 572: Run already exists in server. Run id(s): {8989011}
[2018-04-11 21:27:13] Error in run 574: Run already exists in server. Run id(s): {8989014}
[2018-04-11 21:27:13] Error in run 573: Run already exists in server. Run id(s): {8989016}
[2018-04-11 21:27:13] Started run 580 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:13] Started run 581 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:13] Started run 582 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:13] Error in run 575: Run already exists in server. Run id(s): {8989018}
[2018-04-11 21:27:13] Error in run 576: Run already exists in server. Run id(s): {8989019}
[2018-04-11 21:27:13] Started run 583 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:13] Started run 584 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:16] Error in run 577: Run already exists in server. Run id(s): {8989020}
[2018-04-11 21:27:16] Started run 585 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:16] Error in run 578: Run already exists in server. Run id(s): {8989023}
[2018-04-11 21:27:16] Started run 586 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:17] Error in run 579: Run already exists in server. Run id(s): {8989031}
[2018-04-11 21:27:17] Started run 587 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:17] Error in run 580: Run already exists in server. Run id(s): {8989040}
[2018-04-11 21:27:17] Error in run 581: Run already exists in server. Run id(s): {8989026}
[2018-04-11 21:27:17] Started run 588 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:18] Error in run 582: Run already exists in server. Run id(s): {8989046}
[2018-04-11 21:27:18] Started run 590 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:17] Started run 589 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:18] Error in run 583: Run already exists in server. Run id(s): {8989036}
[2018-04-11 21:27:18] Started run 591 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:18] Error in run 584: Run already exists in server. Run id(s): {8989041}
[2018-04-11 21:27:18] Started run 592 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:20] Error in run 585: Run already exists in server. Run id(s): {8989045}
[2018-04-11 21:27:20] Started run 593 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:21] Error in run 586: Run already exists in server. Run id(s): {8989049}
[2018-04-11 21:27:21] Started run 594 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:21] Error in run 587: Run already exists in server. Run id(s): {8989050}
[2018-04-11 21:27:21] Started run 595 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:22] Error in run 588: Run already exists in server. Run id(s): {8989053}
[2018-04-11 21:27:22] Started run 596 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:22] Error in run 589: Run already exists in server. Run id(s): {8989051}
[2018-04-11 21:27:22] Error in run 590: Run already exists in server. Run id(s): {8989055}
[2018-04-11 21:27:22] Started run 597 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:22] Started run 598 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:22] Error in run 591: Run already exists in server. Run id(s): {8989054}
[2018-04-11 21:27:22] Started run 599 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:23] Error in run 592: Run already exists in server. Run id(s): {8989058}
[2018-04-11 21:27:23] Started run 600 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:25] Error in run 593: Run already exists in server. Run id(s): {8989064}
[2018-04-11 21:27:25] Started run 601 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:26] Error in run 595: Run already exists in server. Run id(s): {8989059}
[2018-04-11 21:27:26] Error in run 594: Run already exists in server. Run id(s): {8989076}
[2018-04-11 21:27:26] Started run 602 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:26] Started run 603 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:27] Error in run 598: Run already exists in server. Run id(s): {8989067}
[2018-04-11 21:27:27] Error in run 596: Run already exists in server. Run id(s): {8989060}
[2018-04-11 21:27:27] Error in run 597: Run already exists in server. Run id(s): {8989080}
[2018-04-11 21:27:27] Started run 604 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:27] Started run 606 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:27] Started run 605 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:27] Error in run 599: Run already exists in server. Run id(s): {8989072}
[2018-04-11 21:27:27] Started run 607 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:27] Error in run 600: Run already exists in server. Run id(s): {8989086}
[2018-04-11 21:27:27] Started run 608 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:29] Error in run 601: Run already exists in server. Run id(s): {8989077}
[2018-04-11 21:27:29] Started run 609 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:30] Error in run 603: Run already exists in server. Run id(s): {8989083}
[2018-04-11 21:27:30] Error in run 602: Run already exists in server. Run id(s): {8989092}
[2018-04-11 21:27:30] Started run 610 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:30] Started run 611 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:31] Error in run 604: Run already exists in server. Run id(s): {8989085}
[2018-04-11 21:27:31] Started run 612 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:31] Error in run 605: Run already exists in server. Run id(s): {8989091}
[2018-04-11 21:27:31] Error in run 606: Run already exists in server. Run id(s): {8989089}
[2018-04-11 21:27:31] Started run 613 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:31] Started run 614 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:31] Error in run 607: Run already exists in server. Run id(s): {8989090}
[2018-04-11 21:27:31] Started run 615 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:32] Error in run 608: Run already exists in server. Run id(s): {8989093}
[2018-04-11 21:27:32] Started run 616 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:34] Error in run 609: Run already exists in server. Run id(s): {8989110}
[2018-04-11 21:27:34] Started run 617 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:35] Error in run 610: Run already exists in server. Run id(s): {8989097}
[2018-04-11 21:27:35] Started run 618 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:35] Error in run 611: Run already exists in server. Run id(s): {8989099}
[2018-04-11 21:27:35] Started run 619 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:36] Error in run 612: Run already exists in server. Run id(s): {8989098}
[2018-04-11 21:27:36] Started run 620 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:36] Error in run 613: Run already exists in server. Run id(s): {8989102, 9015895}
[2018-04-11 21:27:36] Started run 621 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:36] Error in run 614: Run already exists in server. Run id(s): {8989104}
[2018-04-11 21:27:36] Started run 622 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:36] Error in run 615: Run already exists in server. Run id(s): {8989108}
[2018-04-11 21:27:36] Started run 623 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:36] Error in run 616: Run already exists in server. Run id(s): {8989112}
[2018-04-11 21:27:36] Started run 624 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:39] Error in run 617: Run already exists in server. Run id(s): {8989113}
[2018-04-11 21:27:39] Started run 625 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:39] Error in run 618: Run already exists in server. Run id(s): {8989128}
[2018-04-11 21:27:39] Started run 626 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:39] Error in run 619: Run already exists in server. Run id(s): {8989118}
[2018-04-11 21:27:39] Started run 627 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:40] Error in run 620: Run already exists in server. Run id(s): {8989120}
[2018-04-11 21:27:40] Started run 628 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:40] Error in run 621: Run already exists in server. Run id(s): {8989123}
[2018-04-11 21:27:40] Started run 629 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:40] Error in run 622: Run already exists in server. Run id(s): {8989129}
[2018-04-11 21:27:40] Started run 630 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:40] Error in run 623: Run already exists in server. Run id(s): {8989122}
[2018-04-11 21:27:40] Started run 631 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:41] Error in run 624: Run already exists in server. Run id(s): {8989125}
[2018-04-11 21:27:41] Started run 632 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:43] Error in run 625: Run already exists in server. Run id(s): {8989135}
[2018-04-11 21:27:43] Started run 633 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:44] Error in run 626: Run already exists in server. Run id(s): {8989134}
[2018-04-11 21:27:44] Started run 634 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:44] Error in run 627: Run already exists in server. Run id(s): {8989144}
[2018-04-11 21:27:44] Started run 635 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:45] Error in run 628: Run already exists in server. Run id(s): {8989139}
[2018-04-11 21:27:45] Started run 636 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:45] Error in run 629: Run already exists in server. Run id(s): {8989145}
[2018-04-11 21:27:45] Error in run 630: Run already exists in server. Run id(s): {8989146}
[2018-04-11 21:27:45] Started run 637 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:45] Started run 638 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:45] Error in run 631: Run already exists in server. Run id(s): {9015892, 8989150}
[2018-04-11 21:27:45] Started run 639 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:45] Error in run 632: Run already exists in server. Run id(s): {8989159}
[2018-04-11 21:27:45] Started run 640 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:47] Error in run 633: Run already exists in server. Run id(s): {8989156}
[2018-04-11 21:27:47] Started run 641 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:48] Error in run 634: Run already exists in server. Run id(s): {8989158}
[2018-04-11 21:27:48] Started run 642 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:48] Error in run 635: Run already exists in server. Run id(s): {8989161}
[2018-04-11 21:27:48] Started run 643 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:49] Error in run 636: Run already exists in server. Run id(s): {8989166}
[2018-04-11 21:27:49] Started run 644 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:50] Error in run 639: Run already exists in server. Run id(s): {8989169}
[2018-04-11 21:27:50] Error in run 638: Run already exists in server. Run id(s): {8989168}
[2018-04-11 21:27:50] Started run 645 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:50] Started run 646 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:50] Error in run 637: Run already exists in server. Run id(s): {8989165}
[2018-04-11 21:27:50] Started run 647 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:50] Error in run 640: Run already exists in server. Run id(s): {8989170}
[2018-04-11 21:27:50] Started run 648 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:52] Error in run 641: Run already exists in server. Run id(s): {8989171}
[2018-04-11 21:27:52] Started run 649 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:53] Error in run 642: Run already exists in server. Run id(s): {8989172}
[2018-04-11 21:27:53] Started run 650 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:53] Error in run 643: Run already exists in server. Run id(s): {8989173}
[2018-04-11 21:27:53] Started run 651 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:54] Error in run 644: Run already exists in server. Run id(s): {8989185}
[2018-04-11 21:27:54] Started run 652 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:54] Error in run 646: Run already exists in server. Run id(s): {8989181}
[2018-04-11 21:27:54] Started run 653 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:54] Error in run 645: Run already exists in server. Run id(s): {8989182}
[2018-04-11 21:27:54] Started run 654 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:54] Error in run 648: Run already exists in server. Run id(s): {8989187}
[2018-04-11 21:27:54] Error in run 647: Run already exists in server. Run id(s): {8989184}
[2018-04-11 21:27:54] Started run 655 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:54] Started run 656 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:56] Error in run 649: Run already exists in server. Run id(s): {8989188}
[2018-04-11 21:27:56] Started run 657 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:58] Error in run 651: Run already exists in server. Run id(s): {8989191}
[2018-04-11 21:27:58] Error in run 650: Run already exists in server. Run id(s): {8989189}
[2018-04-11 21:27:58] Started run 658 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:58] Started run 659 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:58] Error in run 652: Run already exists in server. Run id(s): {8989195}
[2018-04-11 21:27:58] Started run 660 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:59] Error in run 653: Run already exists in server. Run id(s): {8989194}
[2018-04-11 21:27:59] Started run 661 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:59] Error in run 654: Run already exists in server. Run id(s): {8989197}
[2018-04-11 21:27:59] Started run 662 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:59] Error in run 655: Run already exists in server. Run id(s): {8989198}
[2018-04-11 21:27:59] Started run 663 on task 9952, dataset 'phoneme'.
[2018-04-11 21:27:59] Error in run 656: Run already exists in server. Run id(s): {8989213}
[2018-04-11 21:27:59] Started run 664 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:01] Error in run 657: Run already exists in server. Run id(s): {8989201}
[2018-04-11 21:28:01] Started run 665 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:02] Error in run 658: Run already exists in server. Run id(s): {8989203}
[2018-04-11 21:28:02] Started run 666 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:02] Error in run 659: Run already exists in server. Run id(s): {8989204}
[2018-04-11 21:28:02] Started run 667 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:03] Error in run 660: Run already exists in server. Run id(s): {8989205}
[2018-04-11 21:28:03] Started run 668 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:03] Error in run 661: Run already exists in server. Run id(s): {8989210}
[2018-04-11 21:28:03] Error in run 662: Run already exists in server. Run id(s): {8989208}
[2018-04-11 21:28:03] Started run 669 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:03] Started run 670 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:03] Error in run 664: Run already exists in server. Run id(s): {8989212}
[2018-04-11 21:28:03] Error in run 663: Run already exists in server. Run id(s): {8989211}
[2018-04-11 21:28:03] Started run 671 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:03] Started run 672 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:05] Error in run 665: Run already exists in server. Run id(s): {8989217}
[2018-04-11 21:28:05] Started run 673 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:06] Error in run 666: Run already exists in server. Run id(s): {8989215}
[2018-04-11 21:28:07] Error in run 667: Run already exists in server. Run id(s): {8989219}
[2018-04-11 21:28:06] Started run 674 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:07] Started run 675 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:07] Error in run 668: Run already exists in server. Run id(s): {8989218}
[2018-04-11 21:28:07] Started run 676 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:08] Error in run 670: Run already exists in server. Run id(s): {8989225}
[2018-04-11 21:28:08] Error in run 669: Run already exists in server. Run id(s): {8989224}
[2018-04-11 21:28:08] Started run 677 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:08] Started run 678 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:08] Error in run 671: Run already exists in server. Run id(s): {8989229}
[2018-04-11 21:28:08] Started run 679 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:08] Error in run 672: Run already exists in server. Run id(s): {8989228}
[2018-04-11 21:28:08] Started run 680 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:10] Error in run 673: Run already exists in server. Run id(s): {8989231}
[2018-04-11 21:28:10] Started run 681 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:11] Error in run 674: Run already exists in server. Run id(s): {8989232}
[2018-04-11 21:28:11] Started run 682 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:11] Error in run 675: Run already exists in server. Run id(s): {8989239}
[2018-04-11 21:28:11] Started run 683 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:12] Error in run 676: Run already exists in server. Run id(s): {8989234}
[2018-04-11 21:28:12] Started run 684 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:12] Error in run 679: Run already exists in server. Run id(s): {8989240}
[2018-04-11 21:28:12] Error in run 677: Run already exists in server. Run id(s): {8989236}
[2018-04-11 21:28:12] Started run 685 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:12] Started run 686 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:13] Error in run 678: Run already exists in server. Run id(s): {8989237}
[2018-04-11 21:28:13] Started run 687 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:13] Error in run 680: Run already exists in server. Run id(s): {8989243}
[2018-04-11 21:28:13] Started run 688 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:15] Error in run 681: Run already exists in server. Run id(s): {8989241}
[2018-04-11 21:28:15] Started run 689 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:15] Error in run 682: Run already exists in server. Run id(s): {8989247}
[2018-04-11 21:28:15] Started run 690 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:15] Error in run 683: Run already exists in server. Run id(s): {8989242}
[2018-04-11 21:28:15] Started run 691 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:17] Error in run 684: Run already exists in server. Run id(s): {8989248}
[2018-04-11 21:28:17] Started run 692 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:17] Error in run 686: Run already exists in server. Run id(s): {8989246}
[2018-04-11 21:28:17] Error in run 685: Run already exists in server. Run id(s): {8989250}
[2018-04-11 21:28:17] Started run 693 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:17] Started run 694 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:17] Error in run 688: Run already exists in server. Run id(s): {8989253}
[2018-04-11 21:28:17] Started run 695 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:17] Error in run 687: Run already exists in server. Run id(s): {8989251}
[2018-04-11 21:28:17] Started run 696 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:20] Error in run 689: Run already exists in server. Run id(s): {8989254}
[2018-04-11 21:28:20] Started run 697 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:20] Error in run 690: Run already exists in server. Run id(s): {8989255}
[2018-04-11 21:28:20] Error in run 691: Run already exists in server. Run id(s): {8989260}
[2018-04-11 21:28:20] Started run 698 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:20] Started run 699 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:21] Error in run 692: Run already exists in server. Run id(s): {8989258}
[2018-04-11 21:28:21] Started run 700 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:21] Error in run 694: Run already exists in server. Run id(s): {8989261}
[2018-04-11 21:28:21] Started run 701 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:22] Error in run 693: Run already exists in server. Run id(s): {8989262}
[2018-04-11 21:28:22] Started run 702 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:22] Error in run 695: Run already exists in server. Run id(s): {8989268}
[2018-04-11 21:28:22] Error in run 696: Run already exists in server. Run id(s): {8989272}
[2018-04-11 21:28:22] Started run 703 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:22] Started run 704 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:24] Error in run 697: Run already exists in server. Run id(s): {8989269}
[2018-04-11 21:28:24] Started run 705 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:24] Error in run 699: Run already exists in server. Run id(s): {8989274}
[2018-04-11 21:28:24] Error in run 698: Run already exists in server. Run id(s): {8989279}
[2018-04-11 21:28:24] Started run 706 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:24] Started run 707 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:26] Error in run 700: Run already exists in server. Run id(s): {8989278}
[2018-04-11 21:28:26] Started run 708 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:26] Error in run 701: Run already exists in server. Run id(s): {8989277}
[2018-04-11 21:28:26] Error in run 702: Run already exists in server. Run id(s): {8989280}
[2018-04-11 21:28:26] Started run 709 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:26] Started run 710 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:27] Error in run 703: Run already exists in server. Run id(s): {8989283}
[2018-04-11 21:28:27] Started run 711 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:27] Error in run 704: Run already exists in server. Run id(s): {8989282}
[2018-04-11 21:28:27] Started run 712 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:29] Error in run 705: Run already exists in server. Run id(s): {8989296}
[2018-04-11 21:28:29] Started run 713 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:29] Error in run 707: Run already exists in server. Run id(s): {8989288}
[2018-04-11 21:28:29] Error in run 706: Run already exists in server. Run id(s): {8989286}
[2018-04-11 21:28:29] Started run 714 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:29] Started run 715 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:30] Error in run 708: Run already exists in server. Run id(s): {8989289}
[2018-04-11 21:28:30] Started run 716 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:31] Error in run 710: Run already exists in server. Run id(s): {8989294}
[2018-04-11 21:28:31] Started run 717 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:31] Error in run 709: Run already exists in server. Run id(s): {8989290}
[2018-04-11 21:28:31] Started run 718 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:31] Error in run 711: Run already exists in server. Run id(s): {8989293}
[2018-04-11 21:28:31] Started run 719 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:32] Error in run 712: Run already exists in server. Run id(s): {8989295}
[2018-04-11 21:28:32] Started run 720 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:34] Error in run 713: Run already exists in server. Run id(s): {8989299}
[2018-04-11 21:28:34] Started run 721 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:34] Error in run 714: Run already exists in server. Run id(s): {8989302}
[2018-04-11 21:28:34] Error in run 715: Run already exists in server. Run id(s): {8989305}
[2018-04-11 21:28:34] Started run 722 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:34] Started run 723 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:35] Error in run 716: Run already exists in server. Run id(s): {8989301}
[2018-04-11 21:28:35] Started run 724 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:35] Error in run 718: Run already exists in server. Run id(s): {8989308}
[2018-04-11 21:28:35] Started run 725 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:35] Error in run 717: Run already exists in server. Run id(s): {8989321}
[2018-04-11 21:28:35] Started run 726 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:36] Error in run 719: Run already exists in server. Run id(s): {8989307}
[2018-04-11 21:28:36] Started run 727 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:36] Error in run 720: Run already exists in server. Run id(s): {8989309}
[2018-04-11 21:28:36] Started run 728 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:38] Error in run 721: Run already exists in server. Run id(s): {8989310}
[2018-04-11 21:28:38] Started run 729 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:39] Error in run 723: Run already exists in server. Run id(s): {8989324}
[2018-04-11 21:28:39] Error in run 722: Run already exists in server. Run id(s): {8989311}
[2018-04-11 21:28:39] Started run 730 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:39] Started run 731 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:40] Error in run 724: Run already exists in server. Run id(s): {8989323}
[2018-04-11 21:28:40] Started run 732 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:40] Error in run 725: Run already exists in server. Run id(s): {8989322}
[2018-04-11 21:28:40] Started run 733 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:40] Error in run 726: Run already exists in server. Run id(s): {8989325}
[2018-04-11 21:28:40] Started run 734 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:40] Error in run 727: Run already exists in server. Run id(s): {8989326}
[2018-04-11 21:28:40] Started run 735 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:41] Error in run 728: Run already exists in server. Run id(s): {8989328}
[2018-04-11 21:28:41] Started run 736 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:43] Error in run 729: Run already exists in server. Run id(s): {8989327}
[2018-04-11 21:28:43] Started run 737 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:43] Error in run 731: Run already exists in server. Run id(s): {8989331}
[2018-04-11 21:28:43] Started run 738 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:43] Error in run 730: Run already exists in server. Run id(s): {8989333}
[2018-04-11 21:28:43] Started run 739 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:44] Error in run 732: Run already exists in server. Run id(s): {8989332}
[2018-04-11 21:28:44] Started run 740 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:45] Error in run 733: Run already exists in server. Run id(s): {8989335}
[2018-04-11 21:28:45] Started run 741 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:45] Error in run 735: Run already exists in server. Run id(s): {8989341}
[2018-04-11 21:28:45] Error in run 734: Run already exists in server. Run id(s): {8989339}
[2018-04-11 21:28:45] Started run 742 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:45] Started run 743 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:46] Error in run 736: Run already exists in server. Run id(s): {8989342}
[2018-04-11 21:28:46] Started run 744 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:47] Error in run 737: Run already exists in server. Run id(s): {8989340}
[2018-04-11 21:28:47] Started run 745 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:48] Error in run 739: Run already exists in server. Run id(s): {8989348}
[2018-04-11 21:28:48] Error in run 738: Run already exists in server. Run id(s): {8989345}
[2018-04-11 21:28:48] Started run 746 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:48] Started run 747 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:49] Error in run 740: Run already exists in server. Run id(s): {8989347}
[2018-04-11 21:28:49] Started run 748 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:49] Error in run 741: Run already exists in server. Run id(s): {8989346}
[2018-04-11 21:28:49] Started run 749 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:50] Error in run 743: Run already exists in server. Run id(s): {8989353}
[2018-04-11 21:28:50] Started run 750 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:50] Error in run 742: Run already exists in server. Run id(s): {8989351}
[2018-04-11 21:28:50] Started run 751 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:51] Error in run 744: Run already exists in server. Run id(s): {8989355}
[2018-04-11 21:28:51] Started run 752 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:52] Error in run 745: Run already exists in server. Run id(s): {8989354}
[2018-04-11 21:28:52] Started run 753 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:53] Error in run 747: Run already exists in server. Run id(s): {8989358}
[2018-04-11 21:28:53] Error in run 746: Run already exists in server. Run id(s): {8989360}
[2018-04-11 21:28:53] Started run 754 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:53] Started run 755 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:54] Error in run 749: Run already exists in server. Run id(s): {8989359}
[2018-04-11 21:28:54] Error in run 748: Run already exists in server. Run id(s): {8989362}
[2018-04-11 21:28:54] Started run 756 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:54] Started run 757 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:54] Error in run 750: Run already exists in server. Run id(s): {8989364}
[2018-04-11 21:28:54] Error in run 751: Run already exists in server. Run id(s): {8989366}
[2018-04-11 21:28:54] Started run 758 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:54] Started run 759 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:55] Error in run 752: Run already exists in server. Run id(s): {8989370}
[2018-04-11 21:28:55] Started run 760 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:57] Error in run 753: Run already exists in server. Run id(s): {8989367}
[2018-04-11 21:28:57] Started run 761 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:57] Error in run 754: Run already exists in server. Run id(s): {8989369}
[2018-04-11 21:28:57] Error in run 755: Run already exists in server. Run id(s): {8989374}
[2018-04-11 21:28:57] Started run 762 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:57] Started run 763 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:58] Error in run 756: Run already exists in server. Run id(s): {8989371}
[2018-04-11 21:28:58] Started run 764 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:58] Error in run 757: Run already exists in server. Run id(s): {8989376}
[2018-04-11 21:28:58] Started run 765 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:59] Error in run 758: Run already exists in server. Run id(s): {8989375}
[2018-04-11 21:28:59] Error in run 759: Run already exists in server. Run id(s): {8989377}
[2018-04-11 21:28:59] Started run 766 on task 9952, dataset 'phoneme'.
[2018-04-11 21:28:59] Started run 767 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:00] Error in run 760: Run already exists in server. Run id(s): {8989378}
[2018-04-11 21:29:00] Started run 768 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:01] Error in run 761: Run already exists in server. Run id(s): {8989387}
[2018-04-11 21:29:01] Started run 769 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:02] Error in run 762: Run already exists in server. Run id(s): {8989385}
[2018-04-11 21:29:02] Started run 770 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:02] Error in run 763: Run already exists in server. Run id(s): {8989379}
[2018-04-11 21:29:02] Started run 771 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:03] Error in run 764: Run already exists in server. Run id(s): {8989384}
[2018-04-11 21:29:03] Started run 772 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:03] Error in run 765: Run already exists in server. Run id(s): {8989383}
[2018-04-11 21:29:03] Started run 773 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:03] Error in run 766: Run already exists in server. Run id(s): {8989401}
[2018-04-11 21:29:03] Error in run 767: Run already exists in server. Run id(s): {8989389}
[2018-04-11 21:29:03] Started run 774 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:03] Started run 775 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:04] Error in run 768: Run already exists in server. Run id(s): {8989388}
[2018-04-11 21:29:04] Started run 776 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:06] Error in run 769: Run already exists in server. Run id(s): {8989391}
[2018-04-11 21:29:06] Started run 777 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:07] Error in run 771: Run already exists in server. Run id(s): {8989393}
[2018-04-11 21:29:07] Started run 778 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:07] Error in run 770: Run already exists in server. Run id(s): {8989392}
[2018-04-11 21:29:07] Started run 779 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:07] Error in run 772: Run already exists in server. Run id(s): {8989403}
[2018-04-11 21:29:07] Started run 780 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:07] Error in run 773: Run already exists in server. Run id(s): {8989402}
[2018-04-11 21:29:07] Started run 781 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:08] Error in run 775: Run already exists in server. Run id(s): {8989405}
[2018-04-11 21:29:08] Started run 782 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:08] Error in run 774: Run already exists in server. Run id(s): {8989396}
[2018-04-11 21:29:08] Started run 783 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:09] Error in run 776: Run already exists in server. Run id(s): {8989406}
[2018-04-11 21:29:09] Started run 784 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:10] Error in run 777: Run already exists in server. Run id(s): {8989407}
[2018-04-11 21:29:10] Started run 785 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:11] Error in run 779: Run already exists in server. Run id(s): {8989425}
[2018-04-11 21:29:11] Started run 786 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:11] Error in run 778: Run already exists in server. Run id(s): {8989408}
[2018-04-11 21:29:11] Started run 787 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:12] Error in run 780: Run already exists in server. Run id(s): {8989409}
[2018-04-11 21:29:12] Error in run 781: Run already exists in server. Run id(s): {8989410}
[2018-04-11 21:29:12] Started run 788 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:12] Started run 789 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:13] Error in run 782: Run already exists in server. Run id(s): {8989411}
[2018-04-11 21:29:13] Error in run 783: Run already exists in server. Run id(s): {8989412}
[2018-04-11 21:29:13] Started run 790 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:13] Started run 791 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:14] Error in run 784: Run already exists in server. Run id(s): {8989419}
[2018-04-11 21:29:14] Started run 792 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:15] Error in run 785: Run already exists in server. Run id(s): {8989422}
[2018-04-11 21:29:15] Started run 793 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:16] Error in run 786: Run already exists in server. Run id(s): {8989420}
[2018-04-11 21:29:16] Started run 794 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:16] Error in run 787: Run already exists in server. Run id(s): {8989427}
[2018-04-11 21:29:16] Started run 795 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:17] Error in run 788: Run already exists in server. Run id(s): {8989429}
[2018-04-11 21:29:17] Error in run 789: Run already exists in server. Run id(s): {8989442}
[2018-04-11 21:29:17] Started run 796 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:17] Started run 797 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:17] Error in run 790: Run already exists in server. Run id(s): {8989432}
[2018-04-11 21:29:17] Started run 798 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:18] Error in run 791: Run already exists in server. Run id(s): {8989434}
[2018-04-11 21:29:18] Started run 799 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:18] Error in run 792: Run already exists in server. Run id(s): {8989435}
[2018-04-11 21:29:18] Started run 800 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:19] Error in run 793: Run already exists in server. Run id(s): {8989438}
[2018-04-11 21:29:19] Started run 801 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:20] Error in run 794: Run already exists in server. Run id(s): {8989439}
[2018-04-11 21:29:20] Started run 802 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:21] Error in run 795: Run already exists in server. Run id(s): {8989440}
[2018-04-11 21:29:21] Started run 803 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:21] Error in run 796: Run already exists in server. Run id(s): {8989469}
[2018-04-11 21:29:21] Error in run 797: Run already exists in server. Run id(s): {8989447}
[2018-04-11 21:29:21] Started run 804 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:21] Started run 805 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:22] Error in run 799: Run already exists in server. Run id(s): {8989448}
[2018-04-11 21:29:22] Error in run 798: Run already exists in server. Run id(s): {8989458}
[2018-04-11 21:29:22] Started run 806 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:22] Started run 807 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:23] Error in run 800: Run already exists in server. Run id(s): {8989462}
[2018-04-11 21:29:23] Started run 808 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:24] Error in run 801: Run already exists in server. Run id(s): {8989473}
[2018-04-11 21:29:24] Started run 809 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:25] Error in run 802: Run already exists in server. Run id(s): {8989467}
[2018-04-11 21:29:25] Started run 810 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:25] Error in run 803: Run already exists in server. Run id(s): {8989468}
[2018-04-11 21:29:25] Started run 811 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:26] Error in run 805: Run already exists in server. Run id(s): {8989483}
[2018-04-11 21:29:26] Started run 812 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:26] Error in run 804: Run already exists in server. Run id(s): {8989475}
[2018-04-11 21:29:26] Started run 813 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:27] Error in run 806: Run already exists in server. Run id(s): {8989477}
[2018-04-11 21:29:27] Started run 814 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:27] Error in run 807: Run already exists in server. Run id(s): {8989484}
[2018-04-11 21:29:27] Started run 815 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:27] Error in run 808: Run already exists in server. Run id(s): {8989494}
[2018-04-11 21:29:27] Started run 816 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:29] Error in run 809: Run already exists in server. Run id(s): {8989487}
[2018-04-11 21:29:29] Started run 817 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:29] Error in run 810: Run already exists in server. Run id(s): {8989491}
[2018-04-11 21:29:29] Started run 818 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:30] Error in run 811: Run already exists in server. Run id(s): {8989489}
[2018-04-11 21:29:30] Started run 819 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:31] Error in run 812: Run already exists in server. Run id(s): {8989493}
[2018-04-11 21:29:31] Error in run 813: Run already exists in server. Run id(s): {8989495}
[2018-04-11 21:29:31] Started run 820 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:31] Started run 821 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:32] Error in run 814: Run already exists in server. Run id(s): {8989498}
[2018-04-11 21:29:32] Started run 822 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:32] Error in run 815: Run already exists in server. Run id(s): {8989504}
[2018-04-11 21:29:32] Started run 823 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:32] Error in run 816: Run already exists in server. Run id(s): {8989500}
[2018-04-11 21:29:32] Started run 824 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:33] Error in run 817: Run already exists in server. Run id(s): {8989502}
[2018-04-11 21:29:33] Started run 825 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:34] Error in run 818: Run already exists in server. Run id(s): {8989501}
[2018-04-11 21:29:34] Started run 826 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:34] Error in run 819: Run already exists in server. Run id(s): {8989507}
[2018-04-11 21:29:34] Started run 827 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:36] Error in run 820: Run already exists in server. Run id(s): {8989506}
[2018-04-11 21:29:36] Started run 828 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:36] Error in run 821: Run already exists in server. Run id(s): {8989509}
[2018-04-11 21:29:36] Started run 829 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:36] Error in run 822: Run already exists in server. Run id(s): {8989512}
[2018-04-11 21:29:36] Started run 830 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:36] Error in run 823: Run already exists in server. Run id(s): {8989515}
[2018-04-11 21:29:36] Started run 831 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:36] Error in run 824: Run already exists in server. Run id(s): {8989523}
[2018-04-11 21:29:36] Started run 832 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:38] Error in run 825: Run already exists in server. Run id(s): {8989520}
[2018-04-11 21:29:38] Started run 833 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:38] Error in run 826: Run already exists in server. Run id(s): {8989521}
[2018-04-11 21:29:38] Started run 834 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:39] Error in run 827: Run already exists in server. Run id(s): {8989524}
[2018-04-11 21:29:39] Started run 835 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:40] Error in run 829: Run already exists in server. Run id(s): {8989531}
[2018-04-11 21:29:40] Started run 836 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:40] Error in run 828: Run already exists in server. Run id(s): {8989525}
[2018-04-11 21:29:40] Started run 837 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:41] Error in run 830: Run already exists in server. Run id(s): {8989527}
[2018-04-11 21:29:41] Started run 838 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:41] Error in run 831: Run already exists in server. Run id(s): {8989528}
[2018-04-11 21:29:41] Error in run 832: Run already exists in server. Run id(s): {8989529}
[2018-04-11 21:29:41] Started run 840 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:41] Started run 839 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:42] Error in run 833: Run already exists in server. Run id(s): {8989530}
[2018-04-11 21:29:42] Started run 841 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:43] Error in run 834: Run already exists in server. Run id(s): {8989538}
[2018-04-11 21:29:43] Started run 842 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:44] Error in run 835: Run already exists in server. Run id(s): {8989552}
[2018-04-11 21:29:44] Started run 843 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:45] Error in run 836: Run already exists in server. Run id(s): {8989536}
[2018-04-11 21:29:45] Started run 844 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:45] Error in run 837: Run already exists in server. Run id(s): {8989540}
[2018-04-11 21:29:45] Started run 845 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:45] Error in run 838: Run already exists in server. Run id(s): {8989543}
[2018-04-11 21:29:45] Started run 846 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:45] Error in run 840: Run already exists in server. Run id(s): {8989547}
[2018-04-11 21:29:45] Started run 847 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:45] Error in run 839: Run already exists in server. Run id(s): {8989544}
[2018-04-11 21:29:45] Started run 848 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:47] Error in run 841: Run already exists in server. Run id(s): {8989561}
[2018-04-11 21:29:47] Started run 849 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:48] Error in run 842: Run already exists in server. Run id(s): {8989548}
[2018-04-11 21:29:48] Started run 850 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:48] Error in run 843: Run already exists in server. Run id(s): {8989556}
[2018-04-11 21:29:48] Started run 851 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:49] Error in run 844: Run already exists in server. Run id(s): {8989550}
[2018-04-11 21:29:49] Started run 852 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:49] Error in run 845: Run already exists in server. Run id(s): {8989557}
[2018-04-11 21:29:49] Started run 853 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:50] Error in run 846: Run already exists in server. Run id(s): {8989559}
[2018-04-11 21:29:50] Started run 854 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:50] Error in run 847: Run already exists in server. Run id(s): {8989564}
[2018-04-11 21:29:50] Error in run 848: Run already exists in server. Run id(s): {8989563}
[2018-04-11 21:29:50] Started run 855 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:50] Started run 856 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:52] Error in run 849: Run already exists in server. Run id(s): {8989581}
[2018-04-11 21:29:52] Started run 857 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:52] Error in run 850: Run already exists in server. Run id(s): {8989568}
[2018-04-11 21:29:52] Started run 858 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:53] Error in run 851: Run already exists in server. Run id(s): {8989569}
[2018-04-11 21:29:53] Started run 859 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:54] Error in run 853: Run already exists in server. Run id(s): {8989576}
[2018-04-11 21:29:54] Error in run 852: Run already exists in server. Run id(s): {8989570}
[2018-04-11 21:29:54] Started run 860 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:54] Started run 861 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:54] Error in run 854: Run already exists in server. Run id(s): {8989573}
[2018-04-11 21:29:54] Started run 862 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:55] Error in run 856: Run already exists in server. Run id(s): {8989580}
[2018-04-11 21:29:55] Error in run 855: Run already exists in server. Run id(s): {8989574}
[2018-04-11 21:29:55] Started run 863 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:55] Started run 864 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:56] Error in run 857: Run already exists in server. Run id(s): {8989583}
[2018-04-11 21:29:56] Started run 865 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:57] Error in run 858: Run already exists in server. Run id(s): {8989587}
[2018-04-11 21:29:57] Started run 866 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:57] Error in run 859: Run already exists in server. Run id(s): {8989588}
[2018-04-11 21:29:57] Started run 867 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:59] Error in run 860: Run already exists in server. Run id(s): {8989589}
[2018-04-11 21:29:59] Started run 868 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:59] Error in run 861: Run already exists in server. Run id(s): {8989590}
[2018-04-11 21:29:59] Started run 869 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:59] Error in run 862: Run already exists in server. Run id(s): {8989591}
[2018-04-11 21:29:59] Started run 870 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:59] Error in run 863: Run already exists in server. Run id(s): {8989593}
[2018-04-11 21:29:59] Started run 871 on task 9952, dataset 'phoneme'.
[2018-04-11 21:29:59] Error in run 864: Run already exists in server. Run id(s): {8989592}
[2018-04-11 21:29:59] Started run 872 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:01] Error in run 865: Run already exists in server. Run id(s): {8989595}
[2018-04-11 21:30:01] Started run 873 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:01] Error in run 866: Run already exists in server. Run id(s): {8989594}
[2018-04-11 21:30:01] Started run 874 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:02] Error in run 867: Run already exists in server. Run id(s): {8989607}
[2018-04-11 21:30:02] Started run 875 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:03] Error in run 868: Run already exists in server. Run id(s): {8989596}
[2018-04-11 21:30:03] Started run 876 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:03] Error in run 869: Run already exists in server. Run id(s): {8989597}
[2018-04-11 21:30:03] Started run 877 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:04] Error in run 870: Run already exists in server. Run id(s): {8989602}
[2018-04-11 21:30:04] Started run 878 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:04] Error in run 871: Run already exists in server. Run id(s): {8989605}
[2018-04-11 21:30:04] Started run 879 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:04] Error in run 872: Run already exists in server. Run id(s): {8989603}
[2018-04-11 21:30:04] Started run 880 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:05] Error in run 873: Run already exists in server. Run id(s): {8989609}
[2018-04-11 21:30:05] Started run 881 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:06] Error in run 874: Run already exists in server. Run id(s): {8989610}
[2018-04-11 21:30:06] Started run 882 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:07] Error in run 875: Run already exists in server. Run id(s): {8989613}
[2018-04-11 21:30:07] Started run 883 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:08] Error in run 876: Run already exists in server. Run id(s): {8989626}
[2018-04-11 21:30:08] Started run 884 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:08] Error in run 877: Run already exists in server. Run id(s): {8989614}
[2018-04-11 21:30:08] Started run 885 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:08] Error in run 878: Run already exists in server. Run id(s): {8989615}
[2018-04-11 21:30:08] Started run 886 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:08] Error in run 879: Run already exists in server. Run id(s): {8989616}
[2018-04-11 21:30:08] Started run 887 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:09] Error in run 880: Run already exists in server. Run id(s): {8989617}
[2018-04-11 21:30:09] Started run 888 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:10] Error in run 881: Run already exists in server. Run id(s): {8989618}
[2018-04-11 21:30:10] Started run 889 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:11] Error in run 882: Run already exists in server. Run id(s): {8989625}
[2018-04-11 21:30:11] Started run 890 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:11] Error in run 883: Run already exists in server. Run id(s): {8989623}
[2018-04-11 21:30:11] Started run 891 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:12] Error in run 884: Run already exists in server. Run id(s): {8989638}
[2018-04-11 21:30:12] Started run 892 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:12] Error in run 885: Run already exists in server. Run id(s): {8989631}
[2018-04-11 21:30:12] Started run 893 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:13] Error in run 887: Run already exists in server. Run id(s): {8989656}
[2018-04-11 21:30:13] Started run 894 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:13] Error in run 886: Run already exists in server. Run id(s): {8989636}
[2018-04-11 21:30:13] Error in run 888: Run already exists in server. Run id(s): {8989640}
[2018-04-11 21:30:13] Started run 895 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:13] Started run 896 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:15] Error in run 889: Run already exists in server. Run id(s): {8989645}
[2018-04-11 21:30:15] Started run 897 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:15] Error in run 890: Run already exists in server. Run id(s): {8989654}
[2018-04-11 21:30:15] Started run 898 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:16] Error in run 891: Run already exists in server. Run id(s): {8989648}
[2018-04-11 21:30:16] Started run 899 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:17] Error in run 892: Run already exists in server. Run id(s): {8989650}
[2018-04-11 21:30:17] Error in run 893: Run already exists in server. Run id(s): {8989657}
[2018-04-11 21:30:17] Started run 900 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:17] Started run 901 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:17] Error in run 894: Run already exists in server. Run id(s): {8989658}
[2018-04-11 21:30:17] Started run 902 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:18] Error in run 895: Run already exists in server. Run id(s): {8989659}
[2018-04-11 21:30:18] Started run 903 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:18] Error in run 896: Run already exists in server. Run id(s): {8989681}
[2018-04-11 21:30:18] Started run 904 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:19] Error in run 897: Run already exists in server. Run id(s): {8989660}
[2018-04-11 21:30:19] Started run 905 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:20] Error in run 898: Run already exists in server. Run id(s): {8989661}
[2018-04-11 21:30:20] Started run 906 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:21] Error in run 899: Run already exists in server. Run id(s): {8989665}
[2018-04-11 21:30:21] Started run 907 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:22] Error in run 900: Run already exists in server. Run id(s): {8989668}
[2018-04-11 21:30:22] Started run 908 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:22] Error in run 901: Run already exists in server. Run id(s): {8989670}
[2018-04-11 21:30:22] Started run 909 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:22] Error in run 902: Run already exists in server. Run id(s): {8989682}
[2018-04-11 21:30:22] Started run 910 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:23] Error in run 904: Run already exists in server. Run id(s): {8989679}
[2018-04-11 21:30:23] Error in run 903: Run already exists in server. Run id(s): {8989677}
[2018-04-11 21:30:23] Started run 911 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:23] Started run 912 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:24] Error in run 905: Run already exists in server. Run id(s): {8989685}
[2018-04-11 21:30:24] Started run 913 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:25] Error in run 906: Run already exists in server. Run id(s): {8989687}
[2018-04-11 21:30:25] Started run 914 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:25] Error in run 907: Run already exists in server. Run id(s): {8989689}
[2018-04-11 21:30:25] Started run 915 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:26] Error in run 908: Run already exists in server. Run id(s): {8989690}
[2018-04-11 21:30:26] Started run 916 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:26] Error in run 909: Run already exists in server. Run id(s): {8989691}
[2018-04-11 21:30:27] Error in run 910: Run already exists in server. Run id(s): {8989692}
[2018-04-11 21:30:26] Started run 917 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:27] Started run 918 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:27] Error in run 911: Run already exists in server. Run id(s): {8989694}
[2018-04-11 21:30:27] Error in run 912: Run already exists in server. Run id(s): {8989695}
[2018-04-11 21:30:27] Started run 919 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:27] Started run 920 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:29] Error in run 913: Run already exists in server. Run id(s): {8989698}
[2018-04-11 21:30:29] Started run 921 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:29] Error in run 914: Run already exists in server. Run id(s): {8989701}
[2018-04-11 21:30:29] Started run 922 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:30] Error in run 915: Run already exists in server. Run id(s): {8989706}
[2018-04-11 21:30:30] Started run 923 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:31] Error in run 918: Run already exists in server. Run id(s): {8989705}
[2018-04-11 21:30:31] Error in run 917: Run already exists in server. Run id(s): {8989703}
[2018-04-11 21:30:31] Error in run 916: Run already exists in server. Run id(s): {8989704}
[2018-04-11 21:30:31] Started run 924 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:31] Started run 926 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:31] Started run 925 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:32] Error in run 920: Run already exists in server. Run id(s): {8989712}
[2018-04-11 21:30:32] Started run 927 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:32] Error in run 919: Run already exists in server. Run id(s): {8989717}
[2018-04-11 21:30:32] Started run 928 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:33] Error in run 921: Run already exists in server. Run id(s): {8989713}
[2018-04-11 21:30:33] Started run 929 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:34] Error in run 922: Run already exists in server. Run id(s): {8989715}
[2018-04-11 21:30:34] Started run 930 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:34] Error in run 923: Run already exists in server. Run id(s): {8989722}
[2018-04-11 21:30:34] Started run 931 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:36] Error in run 924: Run already exists in server. Run id(s): {8989720}
[2018-04-11 21:30:36] Error in run 925: Run already exists in server. Run id(s): {8989724}
[2018-04-11 21:30:36] Started run 932 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:36] Error in run 926: Run already exists in server. Run id(s): {8989723}
[2018-04-11 21:30:36] Started run 934 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:36] Started run 933 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:37] Error in run 928: Run already exists in server. Run id(s): {8989735}
[2018-04-11 21:30:37] Started run 935 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:37] Error in run 927: Run already exists in server. Run id(s): {8989726}
[2018-04-11 21:30:37] Started run 936 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:38] Error in run 929: Run already exists in server. Run id(s): {8989734}
[2018-04-11 21:30:38] Started run 937 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:38] Error in run 930: Run already exists in server. Run id(s): {8989736}
[2018-04-11 21:30:38] Started run 938 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:39] Error in run 931: Run already exists in server. Run id(s): {8989738}
[2018-04-11 21:30:39] Started run 939 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:41] Error in run 932: Run already exists in server. Run id(s): {8989744}
[2018-04-11 21:30:41] Started run 940 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:41] Error in run 933: Run already exists in server. Run id(s): {8989745}
[2018-04-11 21:30:41] Started run 941 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:41] Error in run 934: Run already exists in server. Run id(s): {8989746}
[2018-04-11 21:30:41] Started run 942 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:41] Error in run 935: Run already exists in server. Run id(s): {8989748}
[2018-04-11 21:30:41] Started run 943 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:42] Error in run 936: Run already exists in server. Run id(s): {8989756}
[2018-04-11 21:30:42] Started run 944 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:42] Error in run 937: Run already exists in server. Run id(s): {8989750}
[2018-04-11 21:30:42] Started run 945 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:43] Error in run 938: Run already exists in server. Run id(s): {8989749}
[2018-04-11 21:30:43] Started run 946 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:44] Error in run 939: Run already exists in server. Run id(s): {8989755}
[2018-04-11 21:30:44] Started run 947 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:45] Error in run 940: Run already exists in server. Run id(s): {8989754}
[2018-04-11 21:30:45] Started run 948 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:46] Error in run 942: Run already exists in server. Run id(s): {8989758}
[2018-04-11 21:30:46] Started run 949 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:46] Error in run 941: Run already exists in server. Run id(s): {8989757}
[2018-04-11 21:30:46] Started run 950 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:46] Error in run 943: Run already exists in server. Run id(s): {8989762}
[2018-04-11 21:30:46] Started run 951 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:46] Error in run 944: Run already exists in server. Run id(s): {8989767}
[2018-04-11 21:30:46] Started run 952 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:47] Error in run 945: Run already exists in server. Run id(s): {8989768}
[2018-04-11 21:30:47] Started run 953 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:48] Error in run 946: Run already exists in server. Run id(s): {8989773}
[2018-04-11 21:30:48] Started run 954 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:49] Error in run 947: Run already exists in server. Run id(s): {8989770}
[2018-04-11 21:30:49] Started run 955 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:50] Error in run 948: Run already exists in server. Run id(s): {8989771}
[2018-04-11 21:30:50] Started run 956 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:50] Error in run 950: Run already exists in server. Run id(s): {8989772}
[2018-04-11 21:30:50] Started run 957 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:50] Error in run 949: Run already exists in server. Run id(s): {8989778}
[2018-04-11 21:30:50] Started run 958 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:50] Error in run 951: Run already exists in server. Run id(s): {8989775}
[2018-04-11 21:30:50] Started run 959 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:51] Error in run 952: Run already exists in server. Run id(s): {8989776}
[2018-04-11 21:30:51] Started run 960 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:52] Error in run 953: Run already exists in server. Run id(s): {8989782}
[2018-04-11 21:30:52] Started run 961 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:53] Error in run 954: Run already exists in server. Run id(s): {8989783}
[2018-04-11 21:30:53] Started run 962 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:53] Error in run 955: Run already exists in server. Run id(s): {8989787}
[2018-04-11 21:30:53] Started run 963 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:54] Error in run 956: Run already exists in server. Run id(s): {8989784}
[2018-04-11 21:30:54] Started run 964 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:55] Error in run 959: Run already exists in server. Run id(s): {8989791}
[2018-04-11 21:30:55] Error in run 957: Run already exists in server. Run id(s): {8989788}
[2018-04-11 21:30:55] Started run 965 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:55] Error in run 958: Run already exists in server. Run id(s): {8989790}
[2018-04-11 21:30:55] Started run 967 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:55] Started run 966 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:56] Error in run 960: Run already exists in server. Run id(s): {8989793}
[2018-04-11 21:30:56] Started run 968 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:57] Error in run 961: Run already exists in server. Run id(s): {8989794}
[2018-04-11 21:30:57] Started run 969 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:57] Error in run 962: Run already exists in server. Run id(s): {8989795}
[2018-04-11 21:30:57] Started run 970 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:58] Error in run 963: Run already exists in server. Run id(s): {8989796}
[2018-04-11 21:30:58] Started run 971 on task 9952, dataset 'phoneme'.
[2018-04-11 21:30:59] Error in run 964: Run already exists in server. Run id(s): {8989808}
[2018-04-11 21:30:59] Started run 972 on task 9952, dataset 'phoneme'.
[2018-04-11 21:31:00] Error in run 965: Run already exists in server. Run id(s): {8989805}
[2018-04-11 21:31:00] Started run 973 on task 9952, dataset 'phoneme'.
[2018-04-11 21:31:00] Error in run 966: Run already exists in server. Run id(s): {8989799}
[2018-04-11 21:31:00] Error in run 967: Run already exists in server. Run id(s): {8989803}
[2018-04-11 21:31:00] Started run 974 on task 9952, dataset 'phoneme'.
[2018-04-11 21:31:00] Started run 975 on task 9952, dataset 'phoneme'.
[2018-04-11 21:31:01] Error in run 968: Run already exists in server. Run id(s): {8989818}
[2018-04-11 21:31:01] Started run 976 on task 9952, dataset 'phoneme'.
[2018-04-11 21:31:01] Error in run 969: Run already exists in server. Run id(s): {8989806}
[2018-04-11 21:31:01] Started run 977 on task 9952, dataset 'phoneme'.
[2018-04-11 21:31:02] Error in run 970: Run already exists in server. Run id(s): {8989810}
[2018-04-11 21:31:02] Started run 978 on task 9952, dataset 'phoneme'.
[2018-04-11 21:31:03] Error in run 971: Run already exists in server. Run id(s): {8989812}
[2018-04-11 21:31:03] Started run 979 on task 9952, dataset 'phoneme'.
[2018-04-11 21:31:04] Error in run 972: Run already exists in server. Run id(s): {8989813}
[2018-04-11 21:31:04] Started run 980 on task 9952, dataset 'phoneme'.
[2018-04-11 21:31:05] Error in run 973: Run already exists in server. Run id(s): {8989817}
[2018-04-11 21:31:05] Started run 981 on task 9952, dataset 'phoneme'.
[2018-04-11 21:31:05] Error in run 975: Run already exists in server. Run id(s): {8989820}
[2018-04-11 21:31:05] Started run 982 on task 9952, dataset 'phoneme'.
[2018-04-11 21:31:05] Error in run 974: Run already exists in server. Run id(s): {8989819}
[2018-04-11 21:31:05] Started run 983 on task 9952, dataset 'phoneme'.
[2018-04-11 21:31:06] Error in run 976: Run already exists in server. Run id(s): {8989823}
[2018-04-11 21:31:06] Started run 984 on task 9952, dataset 'phoneme'.
[2018-04-11 21:31:06] Error in run 977: Run already exists in server. Run id(s): {8989826}
[2018-04-11 21:31:06] Started run 985 on task 9952, dataset 'phoneme'.
[2018-04-11 21:31:07] Error in run 978: Run already exists in server. Run id(s): {8989824}
[2018-04-11 21:31:07] Started run 986 on task 9952, dataset 'phoneme'.
[2018-04-11 21:31:08] Error in run 979: Run already exists in server. Run id(s): {8989827}
[2018-04-11 21:31:08] Started run 987 on task 9952, dataset 'phoneme'.
[2018-04-11 21:31:09] Error in run 980: Run already exists in server. Run id(s): {8989828}
[2018-04-11 21:31:09] Started run 988 on task 9952, dataset 'phoneme'.
[2018-04-11 21:31:09] Error in run 981: Run already exists in server. Run id(s): {8989832}
[2018-04-11 21:31:09] Started run 989 on task 9952, dataset 'phoneme'.
[2018-04-11 21:31:09] Error in run 983: Run already exists in server. Run id(s): {8989834}
[2018-04-11 21:31:09] Started run 990 on task 9952, dataset 'phoneme'.
[2018-04-11 21:31:09] Error in run 982: Run already exists in server. Run id(s): {8989830}
[2018-04-11 21:31:09] Started run 991 on task 9952, dataset 'phoneme'.
[2018-04-11 21:31:10] Error in run 984: Run already exists in server. Run id(s): {8989833}
[2018-04-11 21:31:10] Started run 992 on task 9952, dataset 'phoneme'.
[2018-04-11 21:31:11] Error in run 985: Run already exists in server. Run id(s): {8989846}
[2018-04-11 21:31:11] Started run 993 on task 9952, dataset 'phoneme'.
[2018-04-11 21:31:12] Error in run 986: Run already exists in server. Run id(s): {8989839}
[2018-04-11 21:31:12] Started run 994 on task 9952, dataset 'phoneme'.
[2018-04-11 21:31:12] Error in run 987: Run already exists in server. Run id(s): {9017209, 8989838}
[2018-04-11 21:31:12] Started run 995 on task 9952, dataset 'phoneme'.
[2018-04-11 21:31:13] Error in run 988: Run already exists in server. Run id(s): {8989840}
[2018-04-11 21:31:13] Started run 996 on task 9952, dataset 'phoneme'.
[2018-04-11 21:31:14] Error in run 989: Run already exists in server. Run id(s): {8989841}
[2018-04-11 21:31:14] Error in run 991: Run already exists in server. Run id(s): {8989845}
[2018-04-11 21:31:14] Started run 997 on task 9952, dataset 'phoneme'.
[2018-04-11 21:31:14] Error in run 990: Run already exists in server. Run id(s): {8989842}
[2018-04-11 21:31:14] Started run 999 on task 9952, dataset 'phoneme'.
[2018-04-11 21:31:14] Started run 998 on task 9952, dataset 'phoneme'.
[2018-04-11 21:31:15] Error in run 992: Run already exists in server. Run id(s): {8989849}
[2018-04-11 21:31:15] Started run 1000 on task 9952, dataset 'phoneme'.
[2018-04-11 21:31:16] Error in run 993: Run already exists in server. Run id(s): {8989850}
[2018-04-11 21:31:17] Error in run 994: Run already exists in server. Run id(s): {8989851}
[2018-04-11 21:31:17] Error in run 995: Run already exists in server. Run id(s): {8989852}
[2018-04-11 21:31:18] Error in run 996: Run already exists in server. Run id(s): {8989854}
[2018-04-11 21:31:19] Error in run 997: Run already exists in server. Run id(s): {8989857}
[2018-04-11 21:31:19] Error in run 998: Run already exists in server. Run id(s): {8989855}
[2018-04-11 21:31:19] Error in run 999: Run already exists in server. Run id(s): {8989860}
[2018-04-11 21:31:20] Error in run 1000: Run already exists in server. Run id(s): {8989859}
